---
title: 'QinHumanGutMicrobial2010'
date: 2022-11-07T21:22:41Z
draft: false
meta_img: "images/image.png"
tags:
  - "one tag"
  - "another tag"
description: "Description for the page"
---
#toImport

# A human gut microbial gene catalog established by metagenomic sequencing
#### (2010) - Junjie Qin, Ruiqiang Li, Jeroen Raes, Manimozhiyan Arumugam, Kristoffer Solvsten Burgdorf, Chaysavanh Manichanh, Trine Nielsen, Nicolas Pons, Florence Levenez, Takuji Yamada, Daniel R. Mende, Junhua Li, Junming Xu, Shaochuan Li, Dongfang Li, Jianjun Cao, Bo Wang, Huiqing Liang, Huisong Zheng, Yinlong Xie, Julien Tap, Patricia Lepage, Marcelo Bertalan, Jean-Michel Batto, Torben Hansen, Denis Le Paslier, Allan Linneberg, H. Bjørn Nielsen, Eric Pelletier, Pierre Renault, Thomas Sicheritz-Ponten, Keith Turner, Hongmei Zhu, Chang Yu, Shengting Li, Min Jian, Yan Zhou, Yingrui Li, Xiuqing Zhang, Songgang Li, Nan Qin, Huanming Yang, Jian Wang, Søren Brunak, Joel Doré, Francisco Guarner, Karsten Kristiansen, Oluf Pedersen, Julian Parkhill, Jean Weissenbach, Peer Bork, S. Dusko Ehrlich, Jun Wang
- **Link**:: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3779803/
- **DOI**:: 10.1038/nature08821
- **Links**:: 
- **Tags**:: #paper
- **Cite Key**:: [@qinHumanGutMicrobial2010] 

read but not much to be used for chicken

### Abstract
To understand the impact of gut microbes on human health and well-being it is crucial to assess their genetic potential. Here we describe the Illumina-based metagenomic sequencing, assembly and characterization of 3.3 million nonredundant microbial genes, derived from 576.7 Gb sequence, from faecal samples of 124 European individuals. The gene set, ~150 times larger than the human gene complement, contains an overwhelming majority of the prevalent microbial genes of the cohort and likely includes a large proportion of the prevalent human intestinal microbial genes. The genes are largely shared among individuals of the cohort. Over 99% of the genes are bacterial, suggesting that the entire cohort harbours between 1000 and 1150 prevalent bacterial species and each individual at least 160 such species, which are also largely shared. We define and describe the minimal gut metagenome and the minimal gut bacterial genome in terms of functions encoded by the gene set.

### Notes
<b>Extracted Annotations (8/29/2022, 2:43:01 PM)</b> 

"www.nature.com/nature" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=1) )

"124 European individuals" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

<i>Un... " this might not be great ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2)</i> )

"It has been estimated that the microbes in our bodies collectively make up to 100 trillion cells, ten-fold the number of human cells, and suggested that they encode 100-fold more unique genes than our own genome" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

<i>Good stat ' ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2)</i> )

"Furthermore, the gut microbes contribute to energy harvest from food, and changes of gut microbiome may be associated with bowel diseases or obesity4-8." ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

"4-8" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

"16S ribosomal RNA gene (rRNA) sequence-based methods9 revealed that two bacterial divisions, the Bacteroidetes and the Firmicutes, constitute over 90% of the known phylogenetic categories and dominate the distal gut microbiota10" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

"10" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

"4,8,10,11" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=2) )

"This gene catalogue contains virtually all of the prevalent gut microbial genes in our cohort, provides a broad view of the functions important for bacterial life in the gut and indicates that many bacterial species are shared by different individuals. Our results also show that short-read metagenomic sequencing can be used for global characterisation of the genetic potential of ecologically complex environments." ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=3) )

"Wanting to generate an extensive catalogue of microbial genes from the human gut, we first assembled the short Illumina reads into longer contigs, which could then be analysed and annotated by standard methods. Using SOAPdenovo19, a de Bruijn graph based tool specially designed for assembling very short reads, we performed de novo assembly for all the Illumina GA sequence data" ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=3) )

<i>Methodology ([Link](zotero://open-pdf/library/items/ZGKDZCRD?page=3)</i>)

