---
title: 'sergeantExtensiveMicrobialFunctional2014'
date: 2022-11-07T21:22:23Z
draft: false
meta_img: "images/image.png"
tags:
  - "one tag"
  - "another tag"
description: "Description for the page"
---
# Extensive Microbial and Functional Diversity within the Chicken Cecal Microbiome
#### (2014) - Martin J. Sergeant, Chrystala Constantinidou, Tristan A. Cogan, Michael R. Bedford, Charles W. Penn, Mark J. Pallen
- **Link**:: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0091941
- **DOI**:: 10.1371/journal.pone.0091941
- **Links**:: 
- **Tags**:: #paper
- **Cite Key**:: [@sergeantExtensiveMicrobialFunctional2014] 

### Abstract
Chickens are major source of food and protein worldwide. Feed conversion and the health of chickens relies on the largely unexplored complex microbial community that inhabits the chicken gut, including the ceca. We have carried out deep microbial community profiling of the microbiota in twenty cecal samples via 16S rRNA gene sequences and an in-depth metagenomics analysis of a single cecal microbiota. We recovered 699 phylotypes, over half of which appear to represent previously unknown species. We obtained 648,251 environmental gene tags (EGTs), the majority of which represent new species. These were binned into over two-dozen draft genomes, which included Campylobacter jejuni and Helicobacter pullorum. We found numerous polysaccharide- and oligosaccharide-degrading enzymes encoding within the metagenome, some of which appeared to be part of polysaccharide utilization systems with genetic evidence for the co-ordination of polysaccharide degradation with sugar transport and utilization. The cecal metagenome encodes several fermentation pathways leading to the production of short-chain fatty acids, including some with novel features. We found a dozen uptake hydrogenases encoded in the metagenome and spe
culate that these provide major hydrogen sinks within this microbial community and might explain the high abundance of several genera within this microbiome, including Campylobacter, Helicobacter and Megamonas.

### Notes

#TODO Re-import once paper is read and annotatted