---
title: 'TiloccaEffectsDietsDifferent2019'
date: 2022-11-07T21:24:42Z
draft: false
meta_img: "images/image.png"
tags:
  - "one tag"
  - "another tag"
description: "Description for the page"
---
# Effects of diets with different phosporus availability on the intestinal microbiota of chickens and pigs
#### (2019) - Bruno Tilocca
- **Link**:: https://www.researchgate.net/publication/331801011_Effects_of_diets_with_different_phosporus_availability_on_the_intestinal_microbiota_of_chickens_and_pigs
- **DOI**:: 
- **Links**:: 
- **Tags**:: #paper
- **Cite Key**:: [@tiloccaEffectsDietsDifferent2019] 

### Abstract
In the research works of the present thesis, 16S rRNA gene sequencing and metaproteomics were employed to investigate the gut microbiota of chickens and pigs kept at experimental diets with varying amount of calcium-phosphorus (CaP) and supplemented MP. This represents a valuable approach to investigate the bacterial specimens involved in the P absorption, allowing for a comprehensive understanding of how the intestinal bacteria adapt to a new diet and which metabolic routes are affected by changing levels of supplemented P and/or MP. Two major experimental trials were performed during the investigation. The first one was conducted on chickens operating a modulation in the dietary levels of Ca, P and MP. This trial highlighted a shift in the composition of the crop and ceca-associated microbial community depending on the composition of the diet fed. Also, investigated protein inventory revealed that the stress condition due to the reduced P availability is mirrored in the gastrointestinal tract (GIT)-associated microbiota. Marked differences were observed in the functions of the bacterial community in the case of P-available diets versus P-deficient ones. Protein repertoire of the first case draws a thriving microbial community focused on complex and anabolic functions. Contrariwise, the bacterial community in the case of P-lacking diets appears to deal with catabolic functions and stress response. The second trial was conducted on pigs and attempts to define the dynamics featuring the microbiota adaptation to a new challenging diet composed of different protein sources and varying levels of Ca and P. Statistical evidences reveal a stepwise adaptation of the fecal microbiota to the experimental diets fed. Both DNA-based approach and metaproteomics independently reveal three main adaptation phases: -before the feeding of the experimental trial (i.e. Zero), -the response of the microbial community to the challenging factor (i.e. MA) and, finally, - the newly achieved homeostatic balance (i.e. EQ). As observed in the first trial, feeding of the experimental diets impairs the overall fecal microbiota composition, stimulating the presence of phase-specific bacterial specimens and a characteristic relative abundance of the shared ones. Bacterial families responsible for the phase-specific architecture of the fecal microbiota are also active in the biochemical pathways driving the functional peculiarities of each adaptation phase. A deeper investigation of the identified protein repertoire revealed that the observed statistical differences among the adaptation phases are uniquely due to the Ca and P composition of the diets fed. None of the observed effects can be attributed to the diverse protein sources supplemented with the diets. Functional categorization of the identified protein inventory depicts three diverse functional assets of the microbial community. Specifically, prior the feeding of the experimental diets, bacteria are hypothesized to live under homeostatic condition, since they appear to be involved in complex and highly-specialized functions. Following the administration of the experimental diets microbial community changes its functional priority and reduce the expression of highly specialized functions to focus on more essential ones. Proteins involved in complex functions such as widening the substrates array and facing complex sugars tend to increase in abundance while the new homeostatic balance is achieved. Altogether, data from both trials provide useful information for future studies aimed to design effective breeding strategies finalized to reduce the P supplementation in the routinely breeding of livestock and maintain a balanced microbial activity in the animal GIT. Investigation of the dynamics of the porcine microbiota provides instructions on the minimal exposure time required from the intestinal microbiota to adapt to a new dietary composition. This is of fundamental importance for the design of future studies aimed to confirm and/or continue our results. Moreover, the anatomical and physiological similarities occurring between humans and pigs, make our findings of interest for future human nutritional studies, where the mechanisms and lasts of the microbiota adaptation process is still object of discussion.

### Notes
<h1>Annotations
 (11/7/2022, 3:08:06 PM)</h1> 

“EFFECTS OF DIETS WITH DIFFERENT PHOSPHORUS AVAILABILITY ON THE INTESTINAL MICROBIOTA OF CHICKENS AND PIGS” (Tilocca, 2019, p. 2) 

“Introduction” (Tilocca, 2019, p. 6) 

“The gut microbiota of chicken” (Tilocca, 2019, p. 6) 

“Methods to characterize the intestinal microorganism” (Tilocca, 2019, p. 6) 

“The microbiota investigation in the “pre-omics era”: Culture-based methods” (Tilocca, 2019, p. 6) 

“Culture-independent methods for studying the gut microbiota” (Tilocca, 2019, p. 6) 

“microbiota investigation in the “omics era” (Tilocca, 2019, p. 6) 

“Project overvie” (Tilocca, 2019, p. 6) 

“Introduction” (Tilocca, 2019, p. 6) 

“The microbiota of chicken” (Tilocca, 2019, p. 6) 

“Concluding remark” (Tilocca, 2019, p. 6) 

“Introductio” (Tilocca, 2019, p. 6) 

“Material and method” (Tilocca, 2019, p. 6) 

“Bioinformatic data analysi” (Tilocca, 2019, p. 6) 

“Chicken proteome...” (Tilocca, 2019, p. 6) 

“Bacterial taxonomy of the protein” (Tilocca, 2019, p. 7) 

“Abundance of metabolic functions varying between the dietary treatments” (Tilocca, 2019, p. 7) 

“The gut microbiota and its importance in animal nutrition...” (Tilocca, 2019, p. 7) 

“Phosphorus and animal nutrition.” (Tilocca, 2019, p. 7) 

“Chicken gut microbiota is affected by the experimental diets fed” (Tilocca, 2019, p. 7) 

“Conclusion.....................................................................................................................134” (Tilocca, 2019, p. 7) 

“Future perspectives” (Tilocca, 2019, p. 7) 

“Phosphorus (P) is the second most abundant mineral in the animal body after calcium (Ca). Approximately 80% of the body P is mineralized as hydroxyapatite, the main mineral composing bones and teeth.” (Tilocca, 2019, p. 13) 

“The remaining 20% of P is commonly found in the internal organs and body fluids where it is involved in a variety of physiological functions of vital importance.” (Tilocca, 2019, p. 13) 

“Indeed, P is a building block for the nucleic acids and regulatory phosphoproteins, hence, of essential importance for the cell growth and differentiation, besides the transmission of the genetic code.” (Tilocca, 2019, p. 13) 

“Inorganic phosphate (PO43-) contributes to maintaining osmotic pressure and the acid-base balance.” (Tilocca, 2019, p. 13) 

“Moreover, P plays a pivotal role in the energetic metabolism and energy transfer via AMP, ADP and ATP with strong implications both at cellular (i.e. glycolysis/gluconeogenesis, protein synthesis, fatty acid transport, Na+/K+ pump) and systemic level (i.e. control of appetite, efficiency of feed utilisation) [1-4].” (Tilocca, 2019, p. 13) 

“Chickens and pigs diets are mainly based on cereals assumption” (Tilocca, 2019, p. 13) 

“In cereals, P is mostly present in the seed coat, where 70-80% of the total phosphorus is stored in the form of phytate (myoinositol 1,2,3,4,5,6-hexakisphosphate, InsP6) [5, 6].” (Tilocca, 2019, p. 13) 

“Plants and cereals are rich in phytases and acid phosphatases, the enzymes involved in the hydrolysis of phytate to low-molecular inositol phosphates (InsP5, InsP4, and InsP3), with the consequent release of one, two, and three PO43- molecules, respectively [9]” (Tilocca, 2019, p. 13) 

“Natural sources of phytases are plants, gut microbes and intestinal epithelial cells. However, abundance and activity of phytase in the latter two is rather reduced in monogastric animals (including chickens and pigs), leading to a reduced bioavailability of the organic phosphorus content in the feedstuff [11]” (Tilocca, 2019, p. 13) So phosphorous is in an inaccessible format so j it just passes through 

“This, in turn, results in a direct excretion of the bound phosphorus through the kidneys along with other bivalent cations complexing the phytate [10, 12].” (Tilocca, 2019, p. 13) 

“Of these, supplementation of microbial phytases in the diet composition is among the most promising strategies, enabling both for increased availability of phosphorus and its reduced renal excretion [10]” (Tilocca, 2019, p. 13) 

“Microbial phytases (MP) are produced intracellularly by Gramnegative bacteria, whereas Gram-positive bacteria and fungi produce these enzymes” (Tilocca, 2019, p. 13) This is cool! 

“extracellularly [9].” (Tilocca, 2019, p. 14) 

“Diet is one of the major environmental factors shaping the composition and activity of the microbial community harboured in the host intestine [13, 14].” (Tilocca, 2019, p. 14) 

“In this context, a large number of diets have been optimized in order to improve diverse aspects of animal husbandry such as stimulation of a beneficial bacterial community and enhance of digestive functions [15]” (Tilocca, 2019, p. 14) 

“The old notion of “organism” intended as a singular entity is nowadays being outclassed, to accept the novel view of the “superorganism” or “metaorganism”.” (Tilocca, 2019, p. 14) Are we sure about this? 

“This innovative perspective integrates the functional districts of the body along with its associated microbial community [16, 17]. The ensemble of microorganisms harbored in a given ecological niche (e.g. the gut) is referred as “microbiota”; whereas, it is defined “microbiome” the microbial community inhabiting a given ecological niche along with their genetic elements and the relationships occurring among members and the environmental habitat where they survive [18].” (Tilocca, 2019, p. 14) Microbiome includes relationships 

“At birth, animals are essentially sterile but immediately after birth all body surfaces (i.e. skin, oral cavity, vagina) and the gastrointestinal tract (GIT) are colonized by an impressive number of microorganisms, including bacteria, archaea, fungi and viruses [19, 20].” (Tilocca, 2019, p. 14) 

“At this stage, the diverse physico-chemical features of each anatomical site enable the maturation of a site-specific microbiota, which composition is the result of the environmental influence and the continuous competition and coexistence between microbial specimens [17].” (Tilocca, 2019, p. 15) 

“The gastrointestinal tract (GIT) microbiota is the most investigated microbial community in both humans and animals. The gut microbiota of adult animals is mainly composed of bacteria, but also virus, fungi, archaea and protozoa are included.” (Tilocca, 2019, p. 15) 

“This huge potential of gene products enables for a wide array of metabolic activities of the intestinal microbiome. Commensal microorganisms colonizing the intestine establish mutualistic relationships that concern a variety of function and physiological processes, including organ development and morphology [24, 25], modulation of the host's immune system [26, 27] and shaping of the metabolic and digestive activity [28, 29].” (Tilocca, 2019, p. 15) 

“The chicken GIT comprises three upper segments (i.e. crop, proventriculus and gizzard) and three lower segments (i.e. small intestine, ceca and large intestine). Crop represent the food” (Tilocca, 2019, p. 15) 

“storage segment and fermentation processes initiate in this organ.” (Tilocca, 2019, p. 16) 

“Feed digestion continues in the proventriculus, also known as the glandular stomach, following the addition of hydrochloric acid and digestive enzymes to the feed.” (Tilocca, 2019, p. 16) 

“Gizzard is the mechanical stomach providing the mashing and mixing of the feed particles.” (Tilocca, 2019, p. 16) 

“The small intestine is further divided into three main sections. Duodenum represents the first section; here, the acidic secretion from the upper gastrointestinal tract are mixed with the bile and pancreatic juices, leading to a sudden alkylation of the environment that marks the beginning of the digestive processes.” (Tilocca, 2019, p. 16) 

“It has been estimated that 95% of the fat digestion occurs in the duodenum [30, 31].” (Tilocca, 2019, p. 16) 

“Jejunal segment extends from the duodenal loop until the Meckel's diverticulum. Main feature of this section is represented by its large size, enabling the digestion and adsorption of the major nutrients, despite a reduced retention time of 40-60 minutes [32-34].” (Tilocca, 2019, p. 16) 

“Ileum is the third segment of the small intestine, it ends at ileo-cecal junction and plays a pivotal role in the absorption of water and minerals. Only a minor activity has been observed in digestion and absorption of fat, carbohydrates and proteins [35].” (Tilocca, 2019, p. 16) 

“Ceca in chickens are two blind sacs that mark the joint point between the small and the large intestine. In these segments, materials of ileal and renal origin are subjected to long retention times, enabling for most of the electrolytes and water absorption [36]. Moreover, the ceca are an important site for the fermentation of any residual raw material with the consequent production of SCFA and B-group vitamins. Ceca empty their content two to three times per day, producing cecal droppings, whose color and texture are used to evaluate the normal functionality of the chickenś digestive tract [36, 37].” (Tilocca, 2019, p. 16) 

“The large intestine represents the last small segment of the digestive tract where occur the reabsorption of the last water content in the fecal material and the digestive wastes are mixed along with the wastes from the urinary system.” (Tilocca, 2019, p. 16) 

“Crop and gizzard share a very similar microbiota, dominated by Lactobacillus spp. and Clostridiaceae. Slight differences are imputable to the reduced fermentation activity of the gizzard reflected as a reduced amount of Enterococcus spp. and Bifidobacteria spp. in its microbiota composition [39].” (Tilocca, 2019, p. 16) 

“Duodenal microbiota is almost exclusively composed of Lactobacillus spp. (up to 99% of the total diversity), but Streptococcus spp. and Enterobacteriaceae may also be abundantly identified [40].” (Tilocca, 2019, p. 16) 

“Despite being dominated by Lactobacillus spp., the ileum is considered as a source of novel bacteria, including butyrate-producing species and other novel bacteria involved in nutrient availability and absorption; Clostridium spp., Streptococcus spp. and Enterococcus” (Tilocca, 2019, p. 16) 

“spp. are also found to dominate the ileal environment [39, 41, 42]” (Tilocca, 2019, p. 17) 

“Cecal microbiota is the most investigated GIT segment since it is an important site of fermentation, enabling the digestion of food rich in cellulose, starch, and resistant polysaccharides. Moreover, the cecum is also the principal site of water reabsorption and nutrient transport and absorption. Therefore, elucidation on the microbial profile harbored in this section provide insights on the mechanisms of food utilization and open new frontiers for the animal husbandry optimization. Ceca is the most colonized section and is featured by a very high bacterial diversity. Here, the most abundant bacterial specimens are Clostridiaceae, Bacteroidaceae, Lactobacillus spp., Proteobacteria, and unknown Firmicutes [39].” (Tilocca, 2019, p. 17) 

“Fig. 2. Distribution of the bacterial families dominating the chicken GIT sections. Section-specific composition of the microbiota has been drawn according to the data published by Videnska et al. [43].” (Tilocca, 2019, p. 17) 

“Videnska et al. [43].” (Tilocca, 2019, p. 17) 

“A handful of studies focused on the characterization of the chicken gut microbiota both in terms of composition and functions. Sergeant and colleagues recently investigated the cecal microbiota using a deep microbial community profiling technique; whilst, information on the potential functions were obtained through metagenomic investigation [44]. Similarly, Tang et al. (2014) investigated the chicken fecal microbiota via 16S rRNA gene sequencing and metaproteomics, elucidating the bacterial” (Tilocca, 2019, p. 17) 

“phylogenetic distribution and the ongoing metabolic processes in the gut of a healthy chicken [45].” (Tilocca, 2019, p. 18) 

“A previous study from Singh et al. (2014) investigated, in a comparative manner, the fecal microbiome of low- and high- feed conversion rate chickens, underlying that genes involved in stress, sulphur assimilation and flagellar motility are overrepresented in low-feed conversion rate animals [46]” (Tilocca, 2019, p. 18) 

“Danzeisen et al. (2011) investigated the chicken cecal microbiome to evaluate the effect of the coccidiostatic monesin and the growth promoters tylosin and virginiamycin. The study provided a first insight on the cecum-associated microbial community and how it is modulated by growth promoters, which is of great importance while developing alternative approaches for animal growth promotion [47].” (Tilocca, 2019, p. 18) 

“Danzeisen” (Tilocca, 2019, p. 18) 

“modulated by growth promoters” (Tilocca, 2019, p. 18) 

“Another recent study performed by Polansky and colleagues investigated the chicken cecal microbiome following inoculation with cecal extracts from chickens of different ages, in order to elucidate the colonization patterns and predict the most promising probiotic genera for cecal colonization of newly hatched chickens [48].” (Tilocca, 2019, p. 18) 

“Besides the above-mentioned studies, the chicken GIT microbiota is also studied in relation to other contexts such as the influence of the dietary intake [49, 50], host genetics [51, 52], gender [51], age [53-55], and spatial microbial diversity [49, 55-57].” (Tilocca, 2019, p. 18) 

“Single strains of microorganisms were cultured in selective medium under controlled environmental conditions (temperature, pH, humidity, % CO2 etc.) to ensure an optimal bacterial growth.” (Tilocca, 2019, p. 21) 

“Cultured microorganisms were subsequently characterized on the basis of genetic and biochemical assays, enabling a broad evaluation of their metabolic capability.” (Tilocca, 2019, p. 21) 

“still today, cultivation attempts are needed to improve our knowledge to both known and unknown specimens” (Tilocca, 2019, p. 22) 

“Although it has been recently demonstrated the potential possibility to cultivate more than 90% of the major gut colonizer through multiple culture conditions [83, 84], these strategies still suffer of the long times required by the microorganisms to growth and their inapplicability to the quantitative assessment of the microbiota composition, due to the cultivation step” (Tilocca, 2019, p. 22) 

“Moreover, phenotypic characterization of the cultured fraction encounters the additional problem that many bacteria, even phylogenetically diverse, share many phenotypic traits, besides their capability of growing in the same selective media and growing conditions [81].” (Tilocca, 2019, p. 22) 

“On the basis of these reasons, the traditional microbiological culture approaches are commonly considered as unsuitable for the study of microbial consortia, such as the case of the gut microbiota, although still in use either as a self-standing method or flanked with other culture-independent methods.” (Tilocca, 2019, p. 22) 

“represented by the community fingerprinting techniques, aimed to provide an overall depict (i.e. fingerprint) of the investigated microbiota [85].” (Tilocca, 2019, p. 22) 

“These techniques include denaturing gradient gel electrophoresis (DGGE), temperature gradient gel electrophoresis (TGGE), and terminal restriction fragment length polymorphism (T-RFLP).” (Tilocca, 2019, p. 22) 

“all the abovementioned techniques share the initial step of 16S rRNA gene amplification [86-88]” (Tilocca, 2019, p. 22) 

“After Woese pioneered the bacterial identification via 16S rRNA gene analysis in the mid-'70s [89], a number of molecular tools facilitating the bacterial identification were developed and, still today, the 16S rRNA gene-based identification of bacteria is adopted for reclassifying previously cultured bacteria or even describe new bacterial species that have never been cultured [90]” (Tilocca, 2019, p. 22) 

“Here, the microbial composition is determined through a direct extraction of the bacterial DNA from the intestinal environment, avoiding the cultivation step, thus the loss of the uncultivable specimens. Universal primers are subsequently adopted to amplify regions of the 16S rRNA gene, creating a clone library of the amplified fragments [81].” (Tilocca, 2019, p. 22) 

“In DGGE studies, 16S rRNA amplicons are separated through a gradient of DNA denaturant (e.g. urea), enabling the separation of DNA fragments of similar size on the basis of their sequence properties. Assuming that every sequence variant represents a different bacterial specimen, the profile produced by this technique indicates the overall taxonomic diversity of the investigated microbiota in a rapid and cost-effective manner [87]. Similar information are also provided by TGGE technique although profile produced by this technique depends on the application of a gradient of temperature (instead of urea) to discriminate 16S rRNA fragments according to their sequence properties [91].” (Tilocca, 2019, p. 23) I've never hard of these methods 

“To this purpose, one of the primer adopted in the 16S rRNA gene amplification must be fluorescently labelled. Marked 16S rDNA fragments are subsequently subjected to restriction through restriction enzymes that cut the DNA sequence at definite recognition sites” (Tilocca, 2019, p. 23) 

“The community profile is finally depicted through a size-by separation of the terminal restriction fragments (i.e. the fluorescently labelled restriction fragments) through capillary electrophoresis. The laser detection acquires the size of the terminal fragments along with the relative fluorescence intensity providing a comprehensive snapshot of the microbiota composition [88, 92].” (Tilocca, 2019, p. 23) 

“their outcome is limited to portray an overall picture of the microbial diversity, and identification of individual microbial species is not supported [82].” (Tilocca, 2019, p. 23) 

“Their main objective is, therefore, the detection and quantification of genomic DNA (genomics), RNA” (Tilocca, 2019, p. 23) 

“transcriptomics), proteins (proteomics) and metabolites (metabolomics) of the studied organism” (Tilocca, 2019, p. 24) 

“instead, by the totality of microorganisms composing the microbiota.” (Tilocca, 2019, p. 24) 

“These community-based approaches are generally referred as metagenomics, metatranscriptomics, metaproteomics, and meta-metabolomics.” (Tilocca, 2019, p. 24) 

“Although considered as single disciplines, none of the single omics science is able to provide a comprehensive and detailed understanding of the biology of the investigated consortia, for which purpose an integrative approach of all these omics sciences is required [82]” (Tilocca, 2019, p. 24) 

“Metagenomics is a community-based approach that employs the study of the microbial genomes collected from the environmental niche where the microorganisms coexist, enabling the study of their phylogenetical, physical and functional features in a culture-independent manner.” (Tilocca, 2019, p. 24) 

“The first approach describes the overall genetic diversity featuring the investigated microbial community by creating a catalogue of the identified genes and genetic elements (e.g. mobile genetic elements)” (Tilocca, 2019, p. 24) 

“The functional analysis of the metagenomic library provides a deep knowledge on the potential function of the microbial community as assessed through the investigation of the functions attributable to the identified genes.” (Tilocca, 2019, p. 24) 

“Moreover, this approach enables discovering of new classes of genes responsible for specific functions and infer novel functions to unannotated genes via sequence homology-based approaches [17, 93, 95].” (Tilocca, 2019, p. 24) 

“Nowadays, 454 Roche pyrosequencing and Illumina sequencing by synthesis are the most frequently adopted techniques [94, 96].” (Tilocca, 2019, p. 24) And now even long sequencing 

“The read length currently achieved with this technology is around 150 bp and nearly 300 bp in the case of the paired-end approach. Multiplexing of the sample is strongly supported by this technology [94, 100, 101].” (Tilocca, 2019, p. 25) 

“Error rate reported in this technology is far smaller than this of 454 Roche. However, the reduced reads length of the Illumina technology resulted in a greater proportion of unassembled reads with important consequences for the functional annotation and/or the neglect of low abundant specimens [102].” (Tilocca, 2019, p. 25) 

“Although featured by the lowest error rate among all NGS technologies, the provided reads cannot exceed 50bp making difficult and unreliable the functional annotation of the reads, but providing interesting results while adopted in resequencing studies [103]” (Tilocca, 2019, p. 25) 

“Other important sequencing technologies are Pac Bio from Pacific Biosciences and Ion Torrent from Life Technologies [94, 96].” (Tilocca, 2019, p. 25) 

“Third Generation Sequencing (TGS). Features of the emerging technology rely on the sequencing of entire single molecules of DNA, avoiding the biases introduced by the step of DNA fragmentation, PCR-based construction of cluster and phased sequencing. Moreover, TGS exploit the high catalytic rates and processivity of the DNA” (Tilocca, 2019, p. 25) 

“polymerase, enabling the sequencing of longer reads in a shorter time, when compared with the “previous” SGS technology.” (Tilocca, 2019, p. 26) 

“TGS technologies are grouped into three main categories depending on the principle exploited to inspect the DNA molecule [104, 105]. Single Molecule Real Time sequencing (SMRT, Pacific Bioscience) is the first TGS technology; it is based on the observance of a DNA polymerase while it synthesizes a ssDNA molecule, using labelled nucleotides [106]. Nanopore sequencing technology inspects the DNA molecule sequence by measuring the translocation of nucleotides across a nanopores-containing membrane. Nucleotides are cleaved by the ssDNA molecule and driven through the nanopore via differential salt concentration [107]. Third category of sequencing technologies relies on the direct observation of the DNA throughout the use of advanced atomic microscopy techniques that enable the visual observation and chemical detection of the DNA molecule building blocks, simultaneously [108].” (Tilocca, 2019, p. 26) 

“Metatranscriptomics is an emerging discipline based on the NGS platforms (mainly Roche and Illumina), aimed to investigate the gene expression of the whole microbial community [93].” (Tilocca, 2019, p. 26) 

“Indeed, metagenomics provides information about all the potential functions a given microbial community is capable of, but does not provide indication on the effective gene expression (or gene dynamics) of the microbial community under determined circumstances [94, 109].” (Tilocca, 2019, p. 26) So it's hypothetical 

“information about all the potential functions a given microbial community is capable of,” (Tilocca, 2019, p. 26) 

“indication on the effective gene expression (or gene dynamics) of the microbial community under determined circumstances” (Tilocca, 2019, p. 26) 

“metatranscriptomics enable a comprehensive annotation of the structure of the transcribed genes, highlighting their start- and stop-site, and assessing the potential splicing variants (or protein isoforms) per each of the expressed gene [110].” (Tilocca, 2019, p. 26) Great for eukaryotes 

“Although the sequencing technology is the same adopted in metagenomics, key differences are represented by the selective removal of the interfering nucleic acids (e.g. DNA, t-RNA, rRNA) and m-RNA to cDNA transcription prior to library production and its subsequent sequencing [111]. A comprehensive review on the RNA-seq technology is provided in [112] and [113].” (Tilocca, 2019, p. 26) 

“Although of enormous importance, metatranscriptomics lack to consider the cellular organization and regulation levels governing the process of mRNA translation and protein maturation (e.g. controlled proteolysis, protein turnover, post-translational modification etc.), which greatly impact the final expression of the genome itself.” (Tilocca, 2019, p. 27) 

“On this basis, one could argue that metaproteomics and metabolomics (addressing the small molecules/metabolites) provide the most realistic picture of the key effectors that directly mediate the metabolic functions operated by the organisms at the specific moment of sampling [115].” (Tilocca, 2019, p. 27) 

“Early proteomics investigations were performed through 2DE, often complemented by spot detection and MS characterization [114]” (Tilocca, 2019, p. 27) 

“An improvement of this technology is represented by 2D-DIGE which employs diverse dyes in a single electrophoresis run, overcoming the issue of the scarce reproducibility among multiple 2DE runs, and enabling a direct comparison of several samples in a high-fidelity manner [114]” (Tilocca, 2019, p. 27) 

“Although these techniques are successful for microbial isolates, interest in the investigation of more complex samples, such as the microbiota, leads to the advent of high-performance MS platforms” (Tilocca, 2019, p. 27) 

“MS is currently the dominant approach for almost all proteomics investigations. It fulfils the technological requirements for a proteomic investigation that include high throughput processing, sensitive analyte detection, wide dynamic range, mass accuracy, ability to characterise and resolve peptide sequences.” (Tilocca, 2019, p. 27) 

“Improvements of the actual MS technologies enable proteomics measurements that provide a wide array of information such as differential proteins expression (e.g. overtime or following experimental treatments), investigation of sub-proteomes (i.e. protein profile of the subcellular structures), post-translational modification (PTM) pattern, protein-protein interactions, and absolute protein quantitation [115].” (Tilocca, 2019, p. 27) 

“MS-based proteomics protocols are mainly distinct into two major branches: top-down- and bottom-up-proteomics. The first is a straightforward strategy, where the complex proteins mixture is separated via liquid chromatography (LC) prior the mass spectrometry (MS) or tandem mass spectrometry (MS/MS) measurements. This protocol is particularly useful when interested in the protein identity but also in its whole molecular form and the presence of isoforms. However, analysis might be severely complicated by the range of possible protein modification (e.g. truncation, SNPs, PTM) affecting the protein mass as predicted from the genome sequence. In addition, the protocol still suffers from drawbacks related to separation and detection of large proteins (&gt;50 kDa). These limitations, coupled with the intrinsic challenge of the metaproteomes (e.g. increased sample complexity, species variability and protein homology), preclude the adoption of the top-down proteomics for the investigation of complex samples such as the microbiota [115, 116].” (Tilocca, 2019, p. 27) 

“Bottom-up (or shotgun) proteomics includes additional processing steps that enable far deeper proteomics measurements. Here, proteins are primarily digested in an enzyme-specific manner (e.g. trypsin) to peptides.” (Tilocca, 2019, p. 28) 

“Peptides are subsequently separated via LC prior to their MS/MS measurements.” (Tilocca, 2019, p. 28) 

“Fragmentation spectrum resulting from the first MS measurement (MS1) indicate the parent ion mass spectrum of the peptides directly eluting from the LC system; whereas the tandem mass spectrum (MS2) provide insights on the fragmentation pattern and sequence information of the previous parent ions. Altogether, mass spectra are acquired and serve as a bar-code that univocally identify a given peptide [117].” (Tilocca, 2019, p. 28) 

“Key point in the experimental proteomics protocols is represented by the efficient protein separation prior to their measurements, either via top-down or bottom-up proteomics” (Tilocca, 2019, p. 28) 

“In this context, gelbased and gel-free proteomics approaches can be distinguished, depending on the use of an intermediate, gel-mediated, pre-fractionation step of the complex proteins mixture.” (Tilocca, 2019, p. 28) 

“Gel-based proteomics is also known as the classical proteomics workflow, expecting a preseparation of the protein mixture through SDS-PAGE or 2DE.” (Tilocca, 2019, p. 28) 

“Nowadays, a variety of offline LC systems, simplifying the complex mixtures upfront the LC-MS/MS instrument, are also employed. These perform the protein separation according to one or multiple dimensions (e.g. isoelectric point, charge, hydrophobicity, size) and are usually compatible with the advanced online separation technologies such as the ultra-high pressure liquid chromatography (UHPLC), providing a better separation of the complex proteins mixture, thus enabling improved protein detection rate [115, 122, 123].” (Tilocca, 2019, p. 28) 

“Metabolomics provides a snapshot of the metabolites suite (metabolome) produced by a given cell, or consortia, at the moment of sampling, enabling their link back to the ongoing physiological processes [124].” (Tilocca, 2019, p. 28) 

“In the context of metabolomics, it is defined “metabolite” any molecule smaller than 1 kDa [125]; this includes intermediates and end-products of metabolic processes, hormones, and other signaling molecules.” (Tilocca, 2019, p. 28) 

“Similarly to transcriptome and proteome, metabolome is very dynamic; therefore, its investigation under diverse circumstances provide precious” (Tilocca, 2019, p. 28) 

“information on the causality of a given phenotype or, inversely, the systemic effects of specific substances (e.g. drugs) and other experimental treatments on the targeted cell lines or even the microbiota [126].” (Tilocca, 2019, p. 29) 

“Main steps in metabolome investigation include the pre-separation of the complex metabolites mixture, following the cell lysis and purification of the extracted metabolites. Metabolites separation is generally accomplished through HPLC, enabling for a wide range of metabolites to be analyzed at a high sensitivity.” (Tilocca, 2019, p. 29) 

“Gold standard for the metabolites detection is represented by MS, where, similarly to proteomics, information on the metabolite IDs and subsequent quantitation is computed back on the basis of the mass spectral fingerprint and the fragmentation pattern. Although HPLC-LC-MS is the most used approach metabolomics, nuclear magnetic resonance (NMR) spectroscopy is also widely used in analysis that specifically targets a reduced number of metabolites [124].” (Tilocca, 2019, p. 29) It would be good to review material from class 

“Metatranscriptomics and metaproteomics are strongly dependent on the quality and availability of the (meta-) genomes, used as a blueprint for the mRNA and protein identification, respectively.” (Tilocca, 2019, p. 29) 

“Although the huge advances in the sequencing technologies, it is still difficult to obtain complete strain-specific genomes and, in the context of the microbiota evaluation, only partial population genomes are generally available.” (Tilocca, 2019, p. 29) 

“Commonly used metagenomics tools are MEGAN (MEtaGenome ANalyzer), IMG/M (Integrated Microbial Genomes/Metagenomes), and MG-RAST (MetaGenome-Rapid Annotation using Subsystem Technology).” (Tilocca, 2019, p. 29) 

“These, although based on different algorithms and a slightly different statistics, enable the taxonomic and functional annotation of the metagenomic sequences, and allow the comparative evaluation of multiple datasets” (Tilocca, 2019, p. 29) 

“A typical LC-MS/MS experiments generate hundreds of thousands of fragmentation spectra. Here, the MS1 provide information on the mass-to-charge (m/z) of the intact peptide along with its amino-acidic composition whereas the MS2, providing information of the fragmentation pattern of each parent ion, determine the correct amino-acidic order.” (Tilocca, 2019, p. 29) 

“The peptide sequences, in” (Tilocca, 2019, p. 29) 

“turn, are assigned to specific protein IDs through a protein database-dependent search, where the measured peptides are compared with the peptides profile deriving from the in silico digestion of user-defined protein database, designated as the reference for the investigated organism or consortia.” (Tilocca, 2019, p. 30) 

“Clearly, this enormous amount of data cannot be manually elaborated and a wide variety of bioinformatics software and tools is nowadays available to perform the computational operations, required to translate the myriad of spectra into a meaningful output that is both concise and informative [115].” (Tilocca, 2019, p. 30) 

“To this purpose, commonly used search engines are OMSSA (Open Mass Spectrometry Search Algorithm), X!Tandem, MASCOT, Andromeda and SEQUEST.” (Tilocca, 2019, p. 30) 

“Major tasks of these tools include quality filtering of the raw MS spectra, peptidespectrum matching and scoring, protein database searching, data mining, and graphical representation of the obtained results [129].” (Tilocca, 2019, p. 30) 

“Similarly to metagenomics tools, proteomics/metaproteomics-committed tools are based on diverse algorithms and statistics, resulting in varying performance that, for each tool, depends on several factors such as sample complexity, separation efficiency and quality of the reference database [127]” (Tilocca, 2019, p. 30) 

“Three main strategies can be employed to obtain a suitable reference database.” (Tilocca, 2019, p. 30) 

“The first option is to use the metagenome sequence of the exact same sample or arising from a comparable community/habitat. This is considered the best option since has been associated with an improved identification rate” (Tilocca, 2019, p. 30) 

“A second choice is to create an artificial, in silico metagenome (pseudometagenome) that includes genome sequences of microbial species supposed to be present in the microbial community of interest. This strategy has been demonstrated to be effective [130], but a priori knowledge of the taxonomic complexity is required.” (Tilocca, 2019, p. 30) 

“The third option expects a survey search before the “official” main search. The survey is conducted against a publicly available database (e.g. NCBInr) and the list of identified proteins is exported either unfiltered or weakly filtered, in order to obtain a comprehensive database to use in the later main search. This approach enables the use of small-sized database (leading to a higher identification rate) and avoids the bias introduced with the manual selection of the microorganisms. Further taxonomic levels may also be implemented from publicly available repositories [127].” (Tilocca, 2019, p. 30) 

“Chapter 1 20 sequences on the basis of the parental ion masses and their fragmentation pattern provided respectively from the MS1 and MS2 of the tandem mass spectrum. This will provide the peptide sequence of peptides that are not included in the current metagenomes as well as enable insights on the PTM and protein polymorphisms. Although of great interest, the accuracy and speed of this groundbreaking approach still need substantial improvements to facilitate its widespread implementation in the “routinely” practice [115, 131].” (Tilocca, 2019, p. 31) This is a wild idea 

“Detailed metaproteomic characterization of a given microbial community requires a quantitative assessment of the identified protein repertoire. The methods applied for protein quantification are distinguished as label-based- or label-free approach depending on the need to label the target proteins to gain quantitative information.” (Tilocca, 2019, p. 31) 

“Because of the independence of any cultivation step and the ability to estimate protein amount without any labelling procedure, the label-free quantification is the most suitable approach for metaproteomic studies [127, 132, 133]. However, this requires high reproducibility LC-MS/MS data.” (Tilocca, 2019, p. 31) 

“Indeed, quantification is based on either spectral counting (SC) or signal intensity (SI). Quantification through SC relies on the assumption that the number of fragment spectra mirrors the abundance of the peptide the spectra belong to. Normalized Spectra Abundance Factor (NSAF) is considered an improved version of the SC, since it estimates the protein abundance considering the effect of the protein size, thus avoiding the overestimation of larger proteins. Main software adopted for SC and NSAF quantification are Scaffold, Proteo IQ, APEX, CENSUS etc; a comprehensive review of these tools is provided in [127, 132, 133].” (Tilocca, 2019, p. 31) 

“The SI-based quantification relies on the peptide ion abundance at specific retention times to estimate protein abundance. Although it is performed by a variety of software (e.g. MaxQuant, Elucidator, Open MS, SIEVE), the label-free quantification provided by MaxQuant is currently the most used, enabling an accurate detection and comparison of changes in the abundance of both high- and low- abundant proteins [127]” (Tilocca, 2019, p. 31) 

“Key points of the MaxQuant software reside in the integrated algorithms performing the process of “ID transfer” and “peak normalization” leading to improved peptide ID and accurate relative protein quantification, respectively [134].” (Tilocca, 2019, p. 31) 

“Another important task of the bioinformatics in the field of metaproteomics concern the annotation of identified proteins into predicted functional classes and clustering of the metaproteome dataset into functional groups.” (Tilocca, 2019, p. 31) 

“Data repositories can be distinguished on the basis of the annotation type. UniProt KB, for instance, is further divided into Swiss-Prot and TrEMBL. The first comprises the only entries whose annotation has been reviewed and manually annotated; whereas, TrEMBL entries are not reviewed and “only” automatically annotated.” (Tilocca, 2019, p. 31) 

“Other data repositories commonly used are Kyoto Encyclopedia of Genes and” (Tilocca, 2019, p. 31) 

“Genomes (KEGG), Gene Ontology (GO), BioCarta” (Tilocca, 2019, p. 32) 

“Moreover, KEGG integrates all these information and organize them as biochemical pathways that fall into seven main groups, namely metabolism, genetic information processing, environmental information processing, cellular process, organismal system, human diseases and drug development.” (Tilocca, 2019, p. 32) 

“Diverse other software and web-based application are nowadays available for this purpose, such as Web MGA, Cytoscape, IPath, DAVID and others (as reviewed in [127]) that retrieve and integrate functional annotation from diverse sequence repositories to provide a more comprehensive functional annotation of the queried metaproteomics dataset.” (Tilocca, 2019, p. 32) 

“The microbial community harboring the GIT of chickens has been investigated mainly in relation to its spatial microbial diversity [56, 135, 136]. A variety of studies is also focused on the investigation of the avian gut microbiota following dietary supplementation of antimicrobials [47, 137], probiotics [50, 138, 139], and high-energy feed formulations [140]. Only a handful of studies investigated the effect of Ca, P and MP supplementation on the chicken gut microbiota composition [141, 142].” (Tilocca, 2019, p. 32) 

“In the first study, diets varying in the amount of supplemented CaP and/or MP were fed to growing broilers, aimed to investigate whether the differences in the diet composition are also reflected in the gut-associated bacterial community, both in terms of composition and activity of the microbial community. A metaproteomic approach was used to assess the dietary-induced changes in the composition of the bacterial fraction of the chickens crop and ceca” (Tilocca, 2019, p. 33) 

“Depicted bacterial community was compared with the total bacterial community designated, for the same sections and dietary treatments, via 16S rRNA gene sequencing. Functional features of the microbial community of chickens crop and ceca identified on protein level, at diverse dietary treatments, were assigned through the functional categorization of the identified protein profiles into bioinformatic data repositories.” (Tilocca, 2019, p. 33) 

“Videnska P, Faldynova M, Juricova H, Babak V, Sisak F, Havlickova H, Rychlik I: Chicken faecal microbiota and disturbances induced by single or repeated therapy with tetracycline and streptomycin. BMC Vet Res 2013, 9(1):30.” (Tilocca, 2019, p. 38) 

“Danzeisen JL, Kim HB, Isaacson RE, Tu ZJ, Johnson TJ: Modulations of the Chicken Cecal Microbiome and Metagenome in Response to Anticoccidial and Growth Promoter Treatment. PloS one 2011, 6(11):e27949.” (Tilocca, 2019, p. 38) 

“Polansky O, Sekelova Z, Faldynova M, Sebkova A, Sisak F, Rychlik I: Important Metabolic Pathways and Biological Processes Expressed by Chicken Cecal Microbiota. Appl Environ Microbiol 2015, 82(5):1569-1576.” (Tilocca, 2019, p. 38) 

“Zoetendal EG, Rajilic-Stojanovic M, de Vos WM: High-throughput diversity and functionality analysis of the gastrointestinal tract microbiota. Gut 2008, 57(11):1605-1615.” (Tilocca, 2019, p. 42) 

“Compared to the structural information obtained by metagenomic analyses, metaproteomic approaches provide, in addition, functional data about the investigated microbiota.” (Tilocca, 2019, p. 49) 

“In general, integration of the main Omics-technologies (genomics, transcriptomics, proteomics and metabolomics) in live science promises highly detailed information about the specific research object and helps to understand molecular changes in response to internal and external environmental factors.” (Tilocca, 2019, p. 49) 

“The microbiotas of livestock like chicken, pig and ruminants are becoming a focus of interest for veterinaries, animal nutritionists and microbiologists. While pig is more often used as an animal model for human-related studies, the rumen microbiota harbors a diversity of enzymes converting complex carbohydrates into monomers which bears high potential for biotechnological applications.” (Tilocca, 2019, p. 49) 

“Despite cultivation being a sound technique to characterize the physiological properties of microorganisms [1], there are severe drawbacks in using this as a tool for characterizing bacterial communities. Typically, the culture media do not resemble in situ conditions and in some cases the carbon richness is higher than the substrates found in situ, allowing the growth of only a small fraction of the community while suppressing other members [2]” (Tilocca, 2019, p. 50) 

“The inconsistency between in situ and cultivable diversity has resulted in the widespread use of culture-independent molecular approaches [3, 4]. Microbial community profiling methods (16S ribosomal RNA gene based approaches) have become important tools to characterize microbial communities and the interactions between the microorganisms present in the GIT.” (Tilocca, 2019, p. 50) 

“Overall, the analysis of the microbiota is important to improve animal nutrition strategies and animal health. This knowledge can be used to modulate the microbiota to reduce antibiotic treatments and, in the case of ruminants, to inhibit the formation of emission gases” (Tilocca, 2019, p. 50) 

“Fig. 1. Workflow of possible methods to study the structure and function of the microbiota in farm animals.” (Tilocca, 2019, p. 51) 

“Both systems have unique features, such as short paired-end reads (max. 2 × 300 bp) with Illumina vs. long read length (600–800 bp) with 454. The latter one is more feasible in terms of shotgun sequencing studies (see below) [5, 6], while Illumina provides barcoding strategies and bigger data sets that are more favorable to analyze hundreds of samples in targeted sequencing projects [6].” (Tilocca, 2019, p. 51) 

“Two other techniques, that were not frequently applied for metagenomic studies of animal microbiota, are the Ion Torrent (Life Technologies) and the PacBio (Pacific biosciences).” (Tilocca, 2019, p. 51) 

“The gene of choice to analyze the phylogenetic composition of a microbial community is the 16S rRNA gene” (Tilocca, 2019, p. 52) 

“There are a couple of techniques in molecular ecology, such as fingerprinting methods, microarrays and fluorescence in situ hybridization which use the 16S rRNA gene as a target molecule.” (Tilocca, 2019, p. 52) 

“Nowadays the total diversity of a microbiological sample is analyzed preferably by pyrosequencing of the 16S rRNA gene, obtained by amplification of extracted DNA.” (Tilocca, 2019, p. 52) 

“Subsequent to pyrosequencing, quality filtering and denoising processes have to be applied. The reads should be checked for chimeras and clustered to operational taxonomic units (OTU) in order to assign the respective taxonomies to the sequences. There is a diverse range of bioinformatic tools available in free software platforms such as Mothur, QIIME, RDP pipeline, LIBSHUFF, UniFrac and MEGAN that support data analysis and convert data to formats that can later be used in statistical packages like R, Metastats or Primer-E. A detailed overview of the methods can be found in several review papers [5, 8].” (Tilocca, 2019, p. 52) 

“overview of the methods can be found in several review papers [5, 8” (Tilocca, 2019, p. 52) 

“These pipelines should be used with special care as it is not only important to make sense of all the raw data, but also to ensure that the final picture is a direct reflection of the original raw data collected and thus of the original community structure of the sample. The output data reveal ecological indices, relative abundance values of the identified taxa and enable a pre-selection for a targeted quantitative PCR (real-time PCR) approaches if necessary.” (Tilocca, 2019, p. 52) 

“In addition to the phylogenetic structure of the community, the analysis of encoded and expressed metabolic pathways is the second objective. Metagenomic or metatranscriptomic data are obtained by shotgun pyrosequencing of the total DNA and cDNA, respectively.” (Tilocca, 2019, p. 52) 

“Reads have to be quality filtered, assembled to contigs, binned and assigned to taxonomies and possible gene functions. As the assembly requires sequence reads with appropriate length, so far 454 pyrosequencing was the method of choice as it produces reads up to 800 bp (see above). Due to the progress in data generation and bioinformatic processing Illumina pyrosequencing is recently used as well. Several tools are available for the annotation of open reading frames on the contigs, MG-RAST [9], MEGAN5 [10], IMG/M [11], Metarep [12] and MicroScope [13]. CAMERA portal [14] was shut down in July 2014. These tools can also be used for metabolic pathway reconstruction. This is usually done based on the KEGG database [15] or the subsystem classification of SEED [16].” (Tilocca, 2019, p. 52) 

“In addition to metatranscriptomic studies, the community activity can be assessed based on expressed proteins and formed metabolites.” (Tilocca, 2019, p. 53) 

“Metaproteomic studies investigate the protein inventory of a specific sample at a certain point of time [17].” (Tilocca, 2019, p. 53) 

“The first key step is to find an optimized sample preparation protocol to avoid co-extraction of eukaryotic proteins and to get a purified protein sample.” (Tilocca, 2019, p. 53) 

“The following workflow depends on the available technical equipment [18]. In a gel-based approach, proteins are separated and proteolytically digested into peptides followed by a one-dimensional liquid chromatography directly coupled to the mass spectrometric analysis (LC–MS/MS).” (Tilocca, 2019, p. 53) 

“In a gel-free approach, peptides are prepared by insolution digestion directly in the protein mix. Peptides are separated by two-dimensional LC and measured by MS/MS analysis.” (Tilocca, 2019, p. 53) 

“The protein identification is the second big challenge as it is highly depending on the available sequence database which can either be used from public resources or sample-specific sequences. An overview of available bioinformatic tools and workflows are given in [18, 19]” (Tilocca, 2019, p. 53) 

“An overview of available bioinformatic tools and workflows are given in [18, 19” (Tilocca, 2019, p. 53) 

“The coverage of metaproteomic studies of complex microbial samples, such as feces or rumen contents, is still low. Since there is a high species diversity and cell density in these types of samples, only abundant proteins are identified while rare species, that may have important metabolic functions, are missed. Targeted proteomic approaches, like selective reaction monitoring (SRM) can be used to specifically detect and quantify proteins of interest [20].” (Tilocca, 2019, p. 53) 

“NMR- and MS-based methods are available and their application to detect defined groups of metabolites is reviewed by Xie et al. [21].” (Tilocca, 2019, p. 53) 

“The chicken intestinal environment comprises a vast and diverse assemblage of microorganisms living not as single species populations, but rather in complex communities comprising multiple species that include animal and human pathogens. Intricate networks of interactions between the microorganisms and their environment shape the respective communities and are important for animal welfare and food safety reasons. The chicken GIT consists of more than 900 species of bacteria. This diverse microbiota helps not only the breakdown and digestion of food but also plays an important role concerning the growth and health of the host [22].” (Tilocca, 2019, p. 53) 

“In the past, the chicken GIT microbial community was studied by culture-based methods. These studies discovered that 10–60% cecal bacteria can be cultured [23, 24] and about 45% could be assigned to the genus level [22]” (Tilocca, 2019, p. 54) 

“The profiles of the different gut sections are nowadays studied using cultivation-independent methods like clone libraries [25-27], denaturing gradient gel electrophoresis (DGGE) [28], temperature gradient electrophoresis (TGGE) [27], terminal restriction fragment length polymorphism (T-RFLP) [4, 29-31], quantitative PCR (qPCR) [32], microarrays [33], next-generation sequencing [34-39] and metaproteomics [40].” (Tilocca, 2019, p. 54) 

“Regarding the research in microbial ecology of the chicken GIT, several studies focused on the influence of diet [25, 31, 32, 41], antimicrobial feed additives [29, 34] host genotype [38, 42], gender [38], spatial microbial diversity [25, 28, 31, 43], age [28], development and temporal microbial variations [22, 26, 28].” (Tilocca, 2019, p. 54) 

“Sklan et al. showed that the different sections of the chicken GIT are highly inter-connected [44]” (Tilocca, 2019, p. 54) 

“However, because of the high diversity within each section, it has been suggested to analyze them as separate ecosystems [28]. It was demonstrated that the microbial communities colonizing the GIT of chicken benefit the host [29, 31, 36]. Nevertheless, two recent studies revealed that this colonization can also harm the host [35, 45].” (Tilocca, 2019, p. 54) 

“This is a moment of great importance regarding the establishment of the microbial communities.” (Tilocca, 2019, p. 54) 

“Although the colonization of the chickens by maternally derived bacteria is low, some studies postulated that the microbial community structure of the small intestine settles within two weeks. Older studies showed that cecal bacteria need longer time to develop [40, 46].” (Tilocca, 2019, p. 54) 

“The gut is colonized by commensal, transient and pathogenic microorganisms. Commensal microorganisms are beneficial to the host as they provide amino acids, short-chain fatty acids and vitamins [40].” (Tilocca, 2019, p. 54) 

“The chicken gut is divided in three upper segments: crop, proventriculus and gizzard. The crop is a food storage muscular pouch related to the breakdown of starch and the fermentation of lactate. Digestion starts in the proventriculus while the gizzard grinds food. Because of its lower pH and fermentation activity, the gizzard functions as microbial barrier. Similar microbial communities were found in the crop and gizzard. Lactobacilli, facultative and microaerophilic bacteria are the most dominant bacteria present in this two segments. Other abundant species belonged to Clostridiaceae, Enterococcus and in the case of the crop also Bifidobacterium and” (Tilocca, 2019, p. 54) 

“Enterobacteriaceae (Fig. 2) [25, 46, 47]” (Tilocca, 2019, p. 55) 

“The small intestine is relatively long and has a constant diameter. It consists of three parts: the duodenum, jejunum and ileum where the nutrient absorption and food digestion occurs” (Tilocca, 2019, p. 55) 

“Due to the low pH, pancreatic and bile secretions, the bacterial density in the duodenum is comparably low” (Tilocca, 2019, p. 55) 

“Besides Lactobacillus as the main colonizer of the jejunum (reaching coverage of up to 99%), Streptococcus was identified as well.” (Tilocca, 2019, p. 55) 

“Amit-Romach et al. has shown that the relative proportion of Lactobacillus spp. in duodenum and jejunum increases within age [48].” (Tilocca, 2019, p. 55) 

“The chicken's ileum harbors Lactobacillus in higher abundance (&gt; 68%) and in lower abundances Streptococcus, Enterobacteriaceae and Clostridiaceae [28, 43].” (Tilocca, 2019, p. 55) 

“Lu et al. demonstrated that during all different stages of microbial community development in the ileum Lactobacilli were dominant [26]. This gut section is also known to be colonized by novel butyrate producing bacteria that may play an important role regarding the availability of nutrients, absorption rate and chicken performance [47].” (Tilocca, 2019, p. 55) 

“Chickens have two caeca which are important for recycling urea, the absorption of water, and digestion of cellulose, starch and polysaccharides. These two fermentation chambers have the highest bacterial density and are colonized by obligate anaerobes like Clostridium, Bacteroidetes, and Bifidobacterium (Fig.2) [42].” (Tilocca, 2019, p. 56) 

“Recently, 16S rDNA amplicon pyrosequencing studies estimated a bacterial population of about 700 species [39]. This wealth of microorganisms makes the caeca an important study site and a reservoir rich in unknown and uncultured microorganisms and pathogens [30, 39, 46, 47].” (Tilocca, 2019, p. 56) 

“Qu et al. proved that mobile DNA elements are the cause of functional microbiome evolution and that horizontal gene transfers and the metavirulomes of cecal microbiomes were related to the host environment [49]. A metagenomic analysis of the chicken caecum using the Illumina MiSeq 2000 system revealed a relatively high proportion of sequences encoding glycosyl hydrolases that were identified by sequence comparison with carbohydrate active enzymes (CAZY) database (Fig. 3) [39].” (Tilocca, 2019, p. 56) 

“More than 200 genes of non-starch polysaccharide degrading enzymes were identified indicating a great potential for xylane degradation compared to a lower cellulolytic potential in the caeca.” (Tilocca, 2019, p. 56) 

“Fig. 3. Abundance of glycoside hydrolase (GH)-families in metagenomes of bovine rumen and chicken caecum. The percentage of each GH-group relative to the total number of GH-families identified in each metagenomic dataset is shown grouped according to major activity [90].” (Tilocca, 2019, p. 57) 

“Chicken feces samples are colonized by Lactobacillus, Clostridium, Faecalibacterium, Ruminococcus, Bacillus, Eubacterium, and Fusobacterium ( Fig. 2)” (Tilocca, 2019, p. 58) 

“Here the microbiota is not stable and it has been proposed that these fluctuations are related to the emptying of the previous gut sections [25]. A recent study in meta-analysis of the avian gut microbiota showed that genes related to cytokine receptors and cell adhesion grouping into “signaling molecules and interaction” were less present in fecal samples indicating a lower potential of host/bacteria interactions [50]” (Tilocca, 2019, p. 58) 

“The only metaproteomic study using a chicken fecal sample identified about 3487 proteins in total [40]. Bacterial proteins mainly belonged to Lactobacillus and Clostridium. Gene ontology analyses showed that the majority encodes for stress-related proteins like chaperons and proteases as well as enzymes involved in glycolysis [40].” (Tilocca, 2019, p. 58) 

“The inclusion of penicillin in diets increases the body weight of chickens and also the Firmicutes to Bacteroidetes ratio in caeca. These effects might be caused by a reduction of the weight of the small intestine and the thickness of the gut wall, increasing the absorption of nutrients” (Tilocca, 2019, p. 58) 

“The addition of the antibiotics tetracycline and streptomycin also induces a rapid shift in microbial community, increasing the prevalence of Lactobacillales and Enterobacteriales in fecal samples.” (Tilocca, 2019, p. 58) 

“For the first time, the present study analysed dietary effects on the protein inventory of the microbiome in crop and ceca of broilers.” (Tilocca, 2019, p. 77) 

“We performed quantitative label-free metaproteomics by using 1D-gel electrophoresis coupled with LC-MS/MS to identify the structural and functional changes triggered by diets supplied with varying amount of mineral phosphorous (P) and microbial phytase (MP).” (Tilocca, 2019, p. 77) 

“Within the ceca section, proteins of Bacteroidaceae were more abundant in the P-supplied diets, whereas proteins of Eubacteriaceae decreased with the P-addition.” (Tilocca, 2019, p. 77) 

“Classification of the identified proteins indicated a thriving microbial community in the case of P and MP supplementation, and stressed microbial community when no P and MP were supplied. Data are available via ProteomeXchange with identifier PXD003805.” (Tilocca, 2019, p. 77) 

“The chicken gastrointestinal tract (GIT) microbiome comprises more than 900 species of microorganisms [1]” (Tilocca, 2019, p. 78) 

“Microbial composition changes longitudinally and radially along the gastrointestinal tract, since each intestinal section has its own characteristic “microenvironment” suitable for a better colonization by specific microbial species [2, 3].” (Tilocca, 2019, p. 78) 

“To date, several studies investigated the chicken microbiota based on cultivation and 16S rRNA gene analysis (for reviews see [2, 4-6]).” (Tilocca, 2019, p. 78) 

“for reviews see [2, 4-6]” (Tilocca, 2019, p. 78) 

“In a recent study, Sergeant et al. [9] analyzed the metagenome of the cecal content from a single bird in order to assess the bacterial phylogenetic distribution and its potential activity. Another recent study investigated the microbiome of two pooled fecal samples based on 16S rRNA gene sequencing and metaproteomics in order to understand the metabolic processes in the gut of a healthy chicken [11].” (Tilocca, 2019, p. 78) 

“Polansky et al. investigated the cecal microbiome of chickens at different ages, including 1-week-old chickens after inoculation with cecal extracts from hens of different ages, on the attempt to explain colonization patterns and predict the most promising probiotic genera for cecal colonization of newly hatched chickens [12].” (Tilocca, 2019, p. 78) 

“Despite the extensive and heterogeneous ensemble of published studies, in-depth investigations about the possible functional changes of the microbiome of the chicken gut challenged with different dietary treatments are missing.” (Tilocca, 2019, p. 78) 

“Phosphorous (P) is an essential macro element involved in a multitude of physiological processes such as bone development, growth and productivity of livestock.” (Tilocca, 2019, p. 78) 

“In plant seeds, organic phosphorous is mainly present as phytate, the salt of myo-inositol 1,2,3,4,5,6-hexakis (dihydrogen phosphate) (phytic acid; InsP6). InsP6 is only partially digestible for many monogastric animals such as chicken [13, 14]. Here, P bioavailability can be improved by phytases, phosphatases responsible for phytic acid dephosphorylation and release of P, available for the animal absorption in the intestine.” (Tilocca, 2019, p. 78) 

“Since endogenous phytases in chicken are less efficient and their activity depends on dietary phosphorous and cholecalciferol [15, 16], standard chicken diet formulation requires supplementation of mineral P and microbial phytase (MP) in order to reduce the problem of low endogenous phytase activity and ensure adequate P” (Tilocca, 2019, p. 78) 

“provision to the animal [17].” (Tilocca, 2019, p. 79) 

“Additional phytase dosage also helps to reduce the need for supplementation of mineral P in the feed formulation.” (Tilocca, 2019, p. 79) 

“This leads to a consequent reduction of P excretion which has a great significance for the ecological problem of water eutrophication and saving of P, a limited resource of global importance.” (Tilocca, 2019, p. 79) 

“To the best of our knowledge, no studies are published so far investigating the functional changes of the bacterial community inhabiting different GIT sections and correlating it to the different effects of broiler diets either supplemented with mineral P and/or MP.” (Tilocca, 2019, p. 79) 

“Witzig and colleagues (2015) characterized changes in bacterial phylogenetic compositions along the GIT of broilers fed with different mineral P and MP supplemented diets by T-RFLP and 16S rRNA gene sequencing [20]. They showed an effect of mineral P and MP towards the abundance of certain Lactobacillus spp. especially in the crop and a possible influence of MP in the cecal community.” (Tilocca, 2019, p. 79) Find a paper to use a as base and add to it 

“Here, we used this feeding experiment to analyze proteins extracted from crop and cecal bacterial communities to obtain indications for possible functional changes in the microbiome of these gut sections. A label-free quantitative (LFQ) metaproteomic approach was used for the assessment of the protein phylogenetic composition and abundance. Discussion on the detected functional pathways and the change of abundance of certain proteins in the diverse dietary treatments is given.” (Tilocca, 2019, p. 79) 

“Samples were obtained from an animal experiment described in detail by Zeller et al. [14]. A schematic overview of the experimental workflow is provided in Figure A in S2 file. A total of 1,140 unsexed broiler hatchlings (Ross 308) were obtained from a local hatchery (Brüterei Süd GmbH and Company KG, Regenstauf, Germany). One hundred and eighty birds were housed in 18 pens (10 animals each) to investigate tibia mineralization, details on [14]. The remaining 960 animals were housed in 48 pens (20 animals each). On day 15 of age, broiler chickens were” (Tilocca, 2019, p. 79) 

“assigned to six experimental treatments (8 pens/treatment) until day 25 of age” (Tilocca, 2019, p. 80) 

“From six pens per treatment, four animal each, were chosen for DNA-based investigation of the chicken GIT microbiota as detailed in [20]. Out of these, 4 animals each from two pens per treatment (48 animals in total) were chosen for the metaproteomic investigations.” (Tilocca, 2019, p. 80) 

“Remaining animals were used for investigation on inositol phosphate degradation [14]. The contents of crop as well as of cecal sections were separately collected and homogenized on a pen basis, yielding two pooled cecal and crop samples, respectively, per dietary treatment, for a total of twenty-four samples (Figure A in S2 file).” (Tilocca, 2019, p. 80) 

“All animals were fed a commercial broiler starter diet until day 14 of age. At day 15 the experimental diets were fed, all consisting of a basal diet [21] containing adequate levels of all nutrients, according to the recommendations of the German Society for Nutritional Physiology (GfE), with the exception of mineral P and calcium” (Tilocca, 2019, p. 80) 

“Three diets contained P exclusively deriving from plant sources (BD–), whereas in another three diets additional P was supplied as mono-calcium phosphate (BD+). Diets of BD – and BD+ groups were further supplemented with 0, 500 and 12500 U/kg respectively of an Escherichia coli microbial phytase product (Quantum Blue, EC 3.1.3.26, supplied by AB Vista, Marlborough, UK), allowing a further distinction between MP0 and MP+ (MP500 and MP12500) diets.” (Tilocca, 2019, p. 80) 

“Twenty four pooled samples were kept on dry ice during their transport to the laboratory and stored at -80°C until their analysis. After thawing at 4°C, bacterial cells were separated by using a previously described method [22] with modifications. Briefly, aliquots of 0.5 g of pooled samples were resuspended by vortexing in 15 mL washing buffer (50 mM Na2HPO4, 0.1% Tween 80, [pH 8.0]). Samples were then incubated for 10 min in a sonication bath (amplitude 50%, 0.5 cycle), shaken for 20 min in a reciprocal shaker at 100 oscillations/min and centrifuged at low speed (200 x g, for 15 min, 4°C). Supernatant containing the bacterial cells was collected in a 50 mL tube and the remaining pellet was subjected to further 4 rounds of the whole protocol, for a total of five rounds.” (Tilocca, 2019, p. 80) 

“Bacterial cells in the pooled supernatant were pelleted by centrifugation at 15,000 x g for 30 min at 4°C and subjected to protein extraction.” (Tilocca, 2019, p. 80) 

“3.3.4 Protein extraction, quantitation, digestion” (Tilocca, 2019, p. 81) 

“MS/MS instrument was set to positive ion mode. Full scan was acquired in the mass range from m/z 300 to 1650 in the Orbitrap mass analyzer at a resolution of 120,000 followed by HCD fragmentation of the 12 most intense precursor ions. High resolution MS/MS spectra were acquired with a resolution of 30,000. The target values were 3*106 charges for the MS scans and 1*105 charges for the MS/MS scans with a maximum fill time of 25 ms and 45 ms,” (Tilocca, 2019, p. 81) 

“respectively. Fragmented masses were excluded for 30 s after MS/MS. Spectra de-noising was performed prior to peptide identification by considering the only top 12 peaks in a window of 100 Da width.” (Tilocca, 2019, p. 82) 

“To reduce the false discovery rate of peptide identification and for a better evaluation of protein abundance a two-step approach for bioinformatics data analysis was chosen. Briefly, proteins identified in the first database-dependent search are used to build a second sample-specific database, used for the second database-dependent search. The smaller size of the latter database enabled for a high coverage of the metaproteome as well as a reduced number of false positive inference [25, 26].” (Tilocca, 2019, p. 82) 

“Acquired raw data were at first processed using Thermo Proteome Discoverer software (v.1.4.1.14), Mascot (v. 2.4) and searched against NCBI-nr bacteria and chicken databases (release July 12th, 2014) in order to export a protein fasta database.” (Tilocca, 2019, p. 82) 

“Methionine oxidation was set as variable modification and carbamidomethylation of cysteine as fixed modification. Default settings of the software were kept, these include protein grouping with peptide confidence set on “high” and delta Cn of 0.1” (Tilocca, 2019, p. 82) 

“In the second process, exported protein fasta file from the first search was subsequently used as in-house database for the peaks alignment and mass re-calibration in the first step of the MaxQuant search. MaxQuant software (v.1.5.1.2) set on LFQ modality was used for peptide identification and protein IDs inference. In the second step of MaxQuant analysis, raw data were independently searched against UniProtKB databases (release October 2014) bacteria (UniProt ID 2, 18976242 entries) and chicken (UniProt ID 9031, 82439 entries).” (Tilocca, 2019, p. 82) 

“The mass spectrometry proteomics data are publicly available in the ProteomeXchange Consortium via the PRIDE [27] partner repository with the dataset identifier PXD003805. Further insights on the identified peptides and their implication for protein IDs inference, are provided in the S2 and S3 Tables.” (Tilocca, 2019, p. 82) 

“Phylogenetic information were inferred on basis of the protein description outputted from the MaxQuant searches. These, in turn, are gathered from the protein annotation of the chosen database (i.e. UniProt KB).” (Tilocca, 2019, p. 83) 

“LFQ abundances from MaxQuantś output results were subjected to statistical analysis by using Primer6 v.6 statistical software (PRIMER-E, Plymouth, UK). Principal coordinate analysis (PCoA) was calculated on the basis of the Bray-Curtis similarity matrix [28].” (Tilocca, 2019, p. 83) 

“Statistical difference between diet treatments was calculated by performing analysis of variance with permutations (PERMANOVA” (Tilocca, 2019, p. 83) 

“Statistically different treatments were then subjected to SIMPER analysis in order to isolate proteins responsible for dissimilarity between pairs of groups with a cut-off threshold of 99.99% [29]” (Tilocca, 2019, p. 83) 

“Selected proteins were functionally classified into COG and KEGG categories by using WebMGA on-line tool [30] with an e-value cutoff of 10-3 and exclusively considering the best hit.” (Tilocca, 2019, p. 83) 

“Cladograms visualizing the dietary effects on the structure of the chickenś crop and cecal microbiome were drawn using the Galaxy on-line tool. It implements the computation of the Linear Discriminant Analysis [21] between the technical triplicates of each experimental diet group (n = 2). Here, the Kruskal-Wallis test is performed to check whether differences between the experimental diets are statistically significant (p &lt;0.05” (Tilocca, 2019, p. 83) 

“Despite using a protocol to analyze microbial proteins, it is a common phenomenon that eukaryotic proteins are always co-extracted and processed during the metaproteomics workflow [33-35]. In this study, a total of 248 and 405 host proteins were identified in the crop and cecal section, respectively, which were investigated in order to highlight the possible presence of specific transporters or any other activity related to the adaptation to the changing dietary conditions.” (Tilocca, 2019, p. 83) 

“Functional classification of the identified proteins was performed by categorization of their abundance intensity into COG classes and KEGG pathways” (Tilocca, 2019, p. 83) 

“No statistical differences in chicken protein abundances were observed between the diets in both GIT sections. The reason may be found in the relatively low protein numbers, which were obtained by the sample preparation protocol, which favored prokaryotic cells [22].” (Tilocca, 2019, p. 84) 

“The used sample preparation protocols yielded a total of 381 and 1,719 bacterial proteins for crop and ceca sections respectively, with 3.1% of the total proteins shared among both sections.” (Tilocca, 2019, p. 84) 

“The relatively low identification rate in the crop section is probably explained by the low bacterial abundance typical of this section and the high amount of feed residues, which were co-extracted [37].” (Tilocca, 2019, p. 84) 

“Details on the number of identified proteins and peptides in each sample, as well as a general overview of the abundance intensities of proteins in both GIT sections along with their grouping into KEGG biochemical pathways, are summarized in Figure B in S2 file and S5 Table.” (Tilocca, 2019, p. 84) 

“In the crop section, 38.9% of the total variation was observed in the PCoA1, where metaproteomes of the samples without mineral P supplementation clustered together and drift apart from the samples supplemented with mineral P (p = 0.043, Fig 1A)” (Tilocca, 2019, p. 84) 

“PCoA analysis of the cecal samples showed a clustering of the microbial proteins from birds fed with mineral P- supplied diets and a separation from treatments without mineral P supplementation, which was not significant (p &gt; 0.05, Fig 1B).” (Tilocca, 2019, p. 84) 

“A very clear distinction was shown between the MP-containing diets that clustered apart from the MPlacking diets (p = 0.008).” (Tilocca, 2019, p. 84) 

“The PCoA analysis of cecal samples (Fig 1B) also showed that the pairs of biological duplicates were close together, with an average similarity of 72.2% (ranging from 69.0% to 84.5%), meaning that observed differences represent a “true” biological difference induced by our experimental treatments.” (Tilocca, 2019, p. 86) 

“The phylogenetic composition of the bacterial proteins extracted from crop and ceca was determined based on the abundance values of the proteins belonging to each bacterial family” (Tilocca, 2019, p. 86) 

“The abundance of each bacterial family in respect to the experimental diets is shown for the crop (Fig 2A) and cecal (Fig 2B) samples” (Tilocca, 2019, p. 86) 

“Only bacterial families showing a discriminative effect (p &lt; 0.05) with respect to the diets were considered and ranked according to their contribution in the different experimental treatments.” (Tilocca, 2019, p. 86) 

“In addition, a phylogenetic assessment was done considering only families expressing at least one phylogenetic marker protein (i.e. highly conserved proteins employed as marker for phylogenetic analysis) [38-41] (S6 Table).” (Tilocca, 2019, p. 86) 

“In general, phylogenetic distribution of the crop metaproteomes showed a reduced bacterial diversity and a high inter-individual diversity among all dietary treatments causing the unpaired scattering of the biological duplicates in the PCoA analysis (Fig 1A)” (Tilocca, 2019, p. 88) 

“In accordance with other studies on the chickenś GIT microbiota composition (for reviews see [2, 4, 6]), proteins belonging to Lactobacillaceae were the most abundant regardless of the diets (Figure C in S2 file).” (Tilocca, 2019, p. 88) 

“The number of proteins belonging to Veillonellaceae increased on average in BD+ diets (36%) when compared with the BD– diets (23%; Figure C panel A in S2 file, Fig 2A).” (Tilocca, 2019, p. 88) 

“Veillonellaceae has been often associated to fiber digestion and short chain fatty acid production [43, 44], therefore its increase in P-supplied diets suggests a potential beneficial effect for the animal growth and performance [45].” (Tilocca, 2019, p. 88) Implied improvement 

“With the exception of Veillonellaceae and a few other families commonly found in the chickenś upper GIT (Propionibacteriaceae, Erysipelotrichaceae, Eubacteriaceae, Clostridiaceae) [46], other minor bacterial families identified in this study such as Nocardiaceae, Gordoniaceae, Bradyrhizobiaceae, Rhizobiaceae, Moraxellaceae, Desulfovibrionaceae and Pseudomonadaceae are more likely to be found in environmental samples [47, 48].” (Tilocca, 2019, p. 88) 

“However, all these families were also found in the gut microbiome of humans or other animals [46, 49-51], therefore their presence in the crop microbiome may be either attributed to an intake from the environment (e.g. with the feed) or such bacterial families can be considered as common members of the cropś microbial fraction.” (Tilocca, 2019, p. 88) 

“The DNA-based phylogenetic analysis of the same crop samples analyzed by Witzig et al. resembled also the predominance of Lactobacillaceae and a decrease of them concomitant to MP supplementation [20]. Other minor bacterial families were either not detected in the DNAbased study or in the present one.” (Tilocca, 2019, p. 88) 

“Phylogenetic distribution of the cecal microbial community (Fig 2B) showed a higher phylogenetic diversity of the identified proteins in all dietary treatments and a change in the composition depending on the diets. Proteins belonging to Bacteroidaceae showed an average abundance of 14.9% in mineral P-supplied diets regardless of the presence of MP compared to 8.4% abundance in the samples from the diets without mineral P supplementation. Conversely, proteins belonging to Eubacteriaceae were more abundant in all diets without mineral P supplementation (4.6%) than with mineral P supplementation (3.9%). Proteins belonging to Ruminococcaceae were more abundant in the diets with increasing amounts of MP (25.4% in the MP+ diets vs. 11.5% in the MP0 diets) whereas proteins of Lactobacillaceae showed a” (Tilocca, 2019, p. 88) 

“contrary abundance (5.2% in MP+ vs. 27.8% in MP0 diets; Fig 2B, Figure C panel B in S2 file). The increased abundance of Bacteroidaceae in the BD+ diets as well as the decrease of Lactobacillaceae due to the MP supplementation is in line with the results of the DNA-based analysis performed by Witzig et al. [20].” (Tilocca, 2019, p. 89) 

“However, no increased abundance of Bacteroidaceae due to the dietary MP addition was observed and no Erysipelotrichaceae members were found in the present metaproteomic investigation.” (Tilocca, 2019, p. 89) 

“Nevertheless, a higher bacterial diversity in the microbiome of crop and ceca sections was highlighted with the present approach. The discrepancy in the results from the same samples is imputable to the different methods. The amplification steps of T-RFLP and amplicon pyrosequencing analyses, including a possible primer bias and the presumed overestimation of taxa with a higher number of 16S rRNA genes, are probably the reason of the reduced heterogeneity found in the microbiota composition” (Tilocca, 2019, p. 89) 

“On the other hand, this step allows a higher sensitivity to target minor bacterial families which are missing in the metaproteomic approach due to the lack of genomic sequences.” (Tilocca, 2019, p. 89) 

“Similarly, inconsistencies between DNA- and protein-based microbiota assessment was described by Tang et al. [11]. They also showed that the correlation between the potential and active bacterial community is not always possible since the species identification in proteomics was different to that of 16S rRNA gene sequencing.” (Tilocca, 2019, p. 89) 

“PERMANOVA analysis of the total datasets of both sections was done to check for significant differences between diets. Crop samples showed that the only pair of treatments MP0 – MP12500 across P factor were statistically significant (p = 0.048).” (Tilocca, 2019, p. 89) 

“The entries of the statistically different treatments were subjected to SIMPER analysis, using a strict cut-off threshold (99.99%), to identify single proteins that caused the dissimilarity between treatment groups. These proteins were functionally classified by grouping them into COG categories (Fig 3).” (Tilocca, 2019, p. 90) 

“In MP12500 metaproteome, lipid metabolism (I) was uniquely identified and three categories were more abundant in comparison with MP0 such as translation, ribosomal structure and biogenesis (J), carbohydrate transport and metabolism (G) and amino acid transport and metabolism (E) (Figure E in S2 file).” (Tilocca, 2019, p. 90) 

“Protein data of the cecal samples showed that proteins belonging to translation, ribosomal structure and biogenesis (J) were more abundant (23%) in the BD- samples than in the BD+ ones (14%) (Fig 3). In contrast, proteins of the carbohydrate transport and metabolism (G) group were more abundant, increasing from 16% in the BD– to 24% in the BD+ diets” (Tilocca, 2019, p. 90) 

“Based on these results, our initial hypotheses that the diets will affect the functional profile of the bacterial communities was accepted. Specifically, mineral P-available diets (BD+; MP+) stimulate a “productive bacterial community” where bacterial resources are focused on complex anabolic functions; while the microbial community present at low mineral P diets (BD– ; MP0) is concerned in the stress response mechanisms, suggesting that the mineral P limitation affecting the host health status is also reflected as stress factor for the gut microbiota. This idea seems to be in accordance with a study of Tang et al., which correlated the expression of proteins involved in metabolic processes of carbohydrate, alcohol and proteins to a thriving microbial community [11]. Protein folding has been linked to the microbial stress response to the high temperature of the chicken body. In our results, the thriving condition is maintained by the adequate nutrients supplementation, while the main stress factor is represented by the lack of P. T” (Tilocca, 2019, p. 91) 

“The metabolic potential of the chickenś fecal microbiota as assessed by Singh et al. [52] shows that the abundance of the “carbohydrate metabolism” SEED subsystem was stable between the metagenomes of low and high feed conversion rate chickens whereas SEED subsystems related to sulphur metabolism and motility/chemotaxis were statistically different.” (Tilocca, 2019, p. 91) 

“The abundance intensities of the proteins, responsible for significant dissimilarities between experimental treatments in the two GIT sections, were additionally grouped according to KEGG biochemical pathways (Figure F in S2 file). In both sections, major biochemical pathways were identified in all experimental treatments, but only a low number of proteins within these pathways was shared between the different treatments, suggesting a diverse overall activity of” (Tilocca, 2019, p. 91) 

“the microbial communities depending on the fed diets” (Tilocca, 2019, p. 92) 

“Additionally, differences in the abundance level of the common proteins across the diets were observed, indicating a probable modulation of the highly conserved functions of the microbiota on the attempt to shape an adequate response to the changing environment. Specifically, the higher abundance of the KEGG Orthologous system [53] for “ribosome” (KO 03010) and “aminoacyl-t-RNA biosynthesis” (KO 00970) observed in crop MP12500 samples, as well as the higher abundance of “ABC transporters” (KO 02010) registered in crop MP0 samples, suggest an increased metabolic activity, and an overall effort of the bacterial community to maximize the P uptake respectively for the MP+ and MP0 samples (Figure F in S2 file). Similar trend was observed in the cecal samples. The high-expression of several KOs such as “glycolysis/gluconeogenesis” (KO 00010, BD+ samples) and “starch and sucrose metabolism” (KO 00500, MP+ samples) indicate a productive bacterial community and a coexistence of the bacterial members in a thriving microenvironment.” (Tilocca, 2019, p. 92) 

“Several KOs were found to be overrepresented in the MP0 treatments such as `pyrimidine metabolism' and `arginine and proline metabolism' (KO 00330). Their abundances decreased with the addition of MP, resulting in a more favorable ratio for the MP0 treatment (Fig 4 panel B and C). Conversely, `lysine biosynthesis' as well as `oxidative phosphorylation' registered an overrepresentation in MP0 in the only pair MP0/MP500, while a further MP addition (MP0/MP12500) led to a slight overrepresentation in the MP12500 sector.” (Tilocca, 2019, p. 92) 

“Identified proteins were sorted into KEGG metabolic maps (S7 Table) in order to confirm the previous COG results and give further support to our hypothesis that experimental diets affect the protein expression of the microbial community” (Tilocca, 2019, p. 93) 

<b>Extracted Annotations (11/7/2022, 3:08:31 PM)</b> 

"EFFECTS OF DIETS WITH DIFFERENT PHOSPHORUS AVAILABILITY ON THE INTESTINAL MICROBIOTA OF CHICKENS AND PIGS" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=2) )

"Introduction" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"The gut microbiota of chicken" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Methods to characterize the intestinal microorganism" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"The microbiota investigation in the "pre-omics era": Culture-based methods" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Culture-independent methods for studying the gut microbiota" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"microbiota investigation in the "omics era" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Project overvie" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Introduction" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"The microbiota of chicken" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Concluding remark" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Introductio" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Material and method" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Bioinformatic data analysi" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Chicken proteome..." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=6) )

"Bacterial taxonomy of the protein" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"Abundance of metabolic functions varying between the dietary treatments" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"The gut microbiota and its importance in animal nutrition..." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"Phosphorus and animal nutrition." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"Chicken gut microbiota is affected by the experimental diets fed" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"Conclusion.....................................................................................................................134" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"Future perspectives" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=7) )

"Phosphorus (P) is the second most abundant mineral in the animal body after calcium (Ca). Approximately 80% of the body P is mineralized as hydroxyapatite, the main mineral composing bones and teeth." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"The remaining 20% of P is commonly found in the internal organs and body fluids where it is involved in a variety of physiological functions of vital importance." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Indeed, P is a building block for the nucleic acids and regulatory phosphoproteins, hence, of essential importance for the cell growth and differentiation, besides the transmission of the genetic code." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Inorganic phosphate (PO43-) contributes to maintaining osmotic pressure and the acid-base balance." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Moreover, P plays a pivotal role in the energetic metabolism and energy transfer via AMP, ADP and ATP with strong implications both at cellular (i.e. glycolysis/gluconeogenesis, protein synthesis, fatty acid transport, Na+/K+ pump) and systemic level (i.e. control of appetite, efficiency of feed utilisation) [1-4]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Chickens and pigs diets are mainly based on cereals assumption" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"In cereals, P is mostly present in the seed coat, where 70-80% of the total phosphorus is stored in the form of phytate (myoinositol 1,2,3,4,5,6-hexakisphosphate, InsP6) [5, 6]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Plants and cereals are rich in phytases and acid phosphatases, the enzymes involved in the hydrolysis of phytate to low-molecular inositol phosphates (InsP5, InsP4, and InsP3), with the consequent release of one, two, and three PO43- molecules, respectively [9]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Natural sources of phytases are plants, gut microbes and intestinal epithelial cells. However, abundance and activity of phytase in the latter two is rather reduced in monogastric animals (including chickens and pigs), leading to a reduced bioavailability of the organic phosphorus content in the feedstuff [11]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

<i>So phosphorous is in an inaccessible format so j it just passes through ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13)</i> )

"This, in turn, results in a direct excretion of the bound phosphorus through the kidneys along with other bivalent cations complexing the phytate [10, 12]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Of these, supplementation of microbial phytases in the diet composition is among the most promising strategies, enabling both for increased availability of phosphorus and its reduced renal excretion [10]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

"Microbial phytases (MP) are produced intracellularly by Gramnegative bacteria, whereas Gram-positive bacteria and fungi produce these enzymes" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13) )

<i>This is cool! ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=13)</i> )

"extracellularly [9]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14) )

"Diet is one of the major environmental factors shaping the composition and activity of the microbial community harboured in the host intestine [13, 14]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14) )

"In this context, a large number of diets have been optimized in order to improve diverse aspects of animal husbandry such as stimulation of a beneficial bacterial community and enhance of digestive functions [15]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14) )

"The old notion of "organism" intended as a singular entity is nowadays being outclassed, to accept the novel view of the "superorganism" or "metaorganism"." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14) )

<i>Are we sure about this? ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14)</i> )

"This innovative perspective integrates the functional districts of the body along with its associated microbial community [16, 17]. The ensemble of microorganisms harbored in a given ecological niche (e.g. the gut) is referred as "microbiota"; whereas, it is defined "microbiome" the microbial community inhabiting a given ecological niche along with their genetic elements and the relationships occurring among members and the environmental habitat where they survive [18]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14) )

<i>Microbiome includes relationships ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14)</i> )

"At birth, animals are essentially sterile but immediately after birth all body surfaces (i.e. skin, oral cavity, vagina) and the gastrointestinal tract (GIT) are colonized by an impressive number of microorganisms, including bacteria, archaea, fungi and viruses [19, 20]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=14) )

"At this stage, the diverse physico-chemical features of each anatomical site enable the maturation of a site-specific microbiota, which composition is the result of the environmental influence and the continuous competition and coexistence between microbial specimens [17]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=15) )

"The gastrointestinal tract (GIT) microbiota is the most investigated microbial community in both humans and animals. The gut microbiota of adult animals is mainly composed of bacteria, but also virus, fungi, archaea and protozoa are included." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=15) )

"This huge potential of gene products enables for a wide array of metabolic activities of the intestinal microbiome. Commensal microorganisms colonizing the intestine establish mutualistic relationships that concern a variety of function and physiological processes, including organ development and morphology [24, 25], modulation of the host's immune system [26, 27] and shaping of the metabolic and digestive activity [28, 29]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=15) )

"The chicken GIT comprises three upper segments (i.e. crop, proventriculus and gizzard) and three lower segments (i.e. small intestine, ceca and large intestine). Crop represent the food" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=15) )

"storage segment and fermentation processes initiate in this organ." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Feed digestion continues in the proventriculus, also known as the glandular stomach, following the addition of hydrochloric acid and digestive enzymes to the feed." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Gizzard is the mechanical stomach providing the mashing and mixing of the feed particles." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"The small intestine is further divided into three main sections. Duodenum represents the first section; here, the acidic secretion from the upper gastrointestinal tract are mixed with the bile and pancreatic juices, leading to a sudden alkylation of the environment that marks the beginning of the digestive processes." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"It has been estimated that 95% of the fat digestion occurs in the duodenum [30, 31]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Jejunal segment extends from the duodenal loop until the Meckel's diverticulum. Main feature of this section is represented by its large size, enabling the digestion and adsorption of the major nutrients, despite a reduced retention time of 40-60 minutes [32-34]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Ileum is the third segment of the small intestine, it ends at ileo-cecal junction and plays a pivotal role in the absorption of water and minerals. Only a minor activity has been observed in digestion and absorption of fat, carbohydrates and proteins [35]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Ceca in chickens are two blind sacs that mark the joint point between the small and the large intestine. In these segments, materials of ileal and renal origin are subjected to long retention times, enabling for most of the electrolytes and water absorption [36]. Moreover, the ceca are an important site for the fermentation of any residual raw material with the consequent production of SCFA and B-group vitamins. Ceca empty their content two to three times per day, producing cecal droppings, whose color and texture are used to evaluate the normal functionality of the chickenś digestive tract [36, 37]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"The large intestine represents the last small segment of the digestive tract where occur the reabsorption of the last water content in the fecal material and the digestive wastes are mixed along with the wastes from the urinary system." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Crop and gizzard share a very similar microbiota, dominated by Lactobacillus spp. and Clostridiaceae. Slight differences are imputable to the reduced fermentation activity of the gizzard reflected as a reduced amount of Enterococcus spp. and Bifidobacteria spp. in its microbiota composition [39]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Duodenal microbiota is almost exclusively composed of Lactobacillus spp. (up to 99% of the total diversity), but Streptococcus spp. and Enterobacteriaceae may also be abundantly identified [40]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"Despite being dominated by Lactobacillus spp., the ileum is considered as a source of novel bacteria, including butyrate-producing species and other novel bacteria involved in nutrient availability and absorption; Clostridium spp., Streptococcus spp. and Enterococcus" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=16) )

"spp. are also found to dominate the ileal environment [39, 41, 42]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=17) )

"Cecal microbiota is the most investigated GIT segment since it is an important site of fermentation, enabling the digestion of food rich in cellulose, starch, and resistant polysaccharides. Moreover, the cecum is also the principal site of water reabsorption and nutrient transport and absorption. Therefore, elucidation on the microbial profile harbored in this section provide insights on the mechanisms of food utilization and open new frontiers for the animal husbandry optimization. Ceca is the most colonized section and is featured by a very high bacterial diversity. Here, the most abundant bacterial specimens are Clostridiaceae, Bacteroidaceae, Lactobacillus spp., Proteobacteria, and unknown Firmicutes [39]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=17) )

"Fig. 2. Distribution of the bacterial families dominating the chicken GIT sections. Section-specific composition of the microbiota has been drawn according to the data published by Videnska et al. [43]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=17) )

"Videnska et al. [43]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=17) )

"A handful of studies focused on the characterization of the chicken gut microbiota both in terms of composition and functions. Sergeant and colleagues recently investigated the cecal microbiota using a deep microbial community profiling technique; whilst, information on the potential functions were obtained through metagenomic investigation [44]. Similarly, Tang et al. (2014) investigated the chicken fecal microbiota via 16S rRNA gene sequencing and metaproteomics, elucidating the bacterial" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=17) )

"phylogenetic distribution and the ongoing metabolic processes in the gut of a healthy chicken [45]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"A previous study from Singh et al. (2014) investigated, in a comparative manner, the fecal microbiome of lowand highfeed conversion rate chickens, underlying that genes involved in stress, sulphur assimilation and flagellar motility are overrepresented in low-feed conversion rate animals [46]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"Danzeisen" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"modulated by growth promoters" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"Danzeisen et al. (2011) investigated the chicken cecal microbiome to evaluate the effect of the coccidiostatic monesin and the growth promoters tylosin and virginiamycin. The study provided a first insight on the cecum-associated microbial community and how it is modulated by growth promoters, which is of great importance while developing alternative approaches for animal growth promotion [47]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"Another recent study performed by Polansky and colleagues investigated the chicken cecal microbiome following inoculation with cecal extracts from chickens of different ages, in order to elucidate the colonization patterns and predict the most promising probiotic genera for cecal colonization of newly hatched chickens [48]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"Besides the above-mentioned studies, the chicken GIT microbiota is also studied in relation to other contexts such as the influence of the dietary intake [49, 50], host genetics [51, 52], gender [51], age [53-55], and spatial microbial diversity [49, 55-57]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=18) )

"Single strains of microorganisms were cultured in selective medium under controlled environmental conditions (temperature, pH, humidity, % CO2 etc.) to ensure an optimal bacterial growth." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=21) )

"Cultured microorganisms were subsequently characterized on the basis of genetic and biochemical assays, enabling a broad evaluation of their metabolic capability." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=21) )

"still today, cultivation attempts are needed to improve our knowledge to both known and unknown specimens" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"Although it has been recently demonstrated the potential possibility to cultivate more than 90% of the major gut colonizer through multiple culture conditions [83, 84], these strategies still suffer of the long times required by the microorganisms to growth and their inapplicability to the quantitative assessment of the microbiota composition, due to the cultivation step" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"Moreover, phenotypic characterization of the cultured fraction encounters the additional problem that many bacteria, even phylogenetically diverse, share many phenotypic traits, besides their capability of growing in the same selective media and growing conditions [81]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"On the basis of these reasons, the traditional microbiological culture approaches are commonly considered as unsuitable for the study of microbial consortia, such as the case of the gut microbiota, although still in use either as a self-standing method or flanked with other culture-independent methods." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"represented by the community fingerprinting techniques, aimed to provide an overall depict (i.e. fingerprint) of the investigated microbiota [85]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"These techniques include denaturing gradient gel electrophoresis (DGGE), temperature gradient gel electrophoresis (TGGE), and terminal restriction fragment length polymorphism (T-RFLP)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"all the abovementioned techniques share the initial step of 16S rRNA gene amplification [86-88]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"After Woese pioneered the bacterial identification via 16S rRNA gene analysis in the mid-'70s [89], a number of molecular tools facilitating the bacterial identification were developed and, still today, the 16S rRNA gene-based identification of bacteria is adopted for reclassifying previously cultured bacteria or even describe new bacterial species that have never been cultured [90]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"Here, the microbial composition is determined through a direct extraction of the bacterial DNA from the intestinal environment, avoiding the cultivation step, thus the loss of the uncultivable specimens. Universal primers are subsequently adopted to amplify regions of the 16S rRNA gene, creating a clone library of the amplified fragments [81]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=22) )

"In DGGE studies, 16S rRNA amplicons are separated through a gradient of DNA denaturant (e.g. urea), enabling the separation of DNA fragments of similar size on the basis of their sequence properties. Assuming that every sequence variant represents a different bacterial specimen, the profile produced by this technique indicates the overall taxonomic diversity of the investigated microbiota in a rapid and cost-effective manner [87]. Similar information are also provided by TGGE technique although profile produced by this technique depends on the application of a gradient of temperature (instead of urea) to discriminate 16S rRNA fragments according to their sequence properties [91]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=23) )

<i>I've never hard of these methods ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=23)</i> )

"To this purpose, one of the primer adopted in the 16S rRNA gene amplification must be fluorescently labelled. Marked 16S rDNA fragments are subsequently subjected to restriction through restriction enzymes that cut the DNA sequence at definite recognition sites" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=23) )

"The community profile is finally depicted through a size-by separation of the terminal restriction fragments (i.e. the fluorescently labelled restriction fragments) through capillary electrophoresis. The laser detection acquires the size of the terminal fragments along with the relative fluorescence intensity providing a comprehensive snapshot of the microbiota composition [88, 92]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=23) )

"their outcome is limited to portray an overall picture of the microbial diversity, and identification of individual microbial species is not supported [82]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=23) )

"Their main objective is, therefore, the detection and quantification of genomic DNA (genomics), RNA" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=23) )

"transcriptomics), proteins (proteomics) and metabolites (metabolomics) of the studied organism" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"instead, by the totality of microorganisms composing the microbiota." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"These community-based approaches are generally referred as metagenomics, metatranscriptomics, metaproteomics, and meta-metabolomics." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"Although considered as single disciplines, none of the single omics science is able to provide a comprehensive and detailed understanding of the biology of the investigated consortia, for which purpose an integrative approach of all these omics sciences is required [82]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"Metagenomics is a community-based approach that employs the study of the microbial genomes collected from the environmental niche where the microorganisms coexist, enabling the study of their phylogenetical, physical and functional features in a culture-independent manner." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"The first approach describes the overall genetic diversity featuring the investigated microbial community by creating a catalogue of the identified genes and genetic elements (e.g. mobile genetic elements)" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"The functional analysis of the metagenomic library provides a deep knowledge on the potential function of the microbial community as assessed through the investigation of the functions attributable to the identified genes." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"Moreover, this approach enables discovering of new classes of genes responsible for specific functions and infer novel functions to unannotated genes via sequence homology-based approaches [17, 93, 95]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

"Nowadays, 454 Roche pyrosequencing and Illumina sequencing by synthesis are the most frequently adopted techniques [94, 96]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24) )

<i>And now even long sequencing ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=24)</i> )

"The read length currently achieved with this technology is around 150 bp and nearly 300 bp in the case of the paired-end approach. Multiplexing of the sample is strongly supported by this technology [94, 100, 101]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=25) )

"Error rate reported in this technology is far smaller than this of 454 Roche. However, the reduced reads length of the Illumina technology resulted in a greater proportion of unassembled reads with important consequences for the functional annotation and/or the neglect of low abundant specimens [102]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=25) )

"Although featured by the lowest error rate among all NGS technologies, the provided reads cannot exceed 50bp making difficult and unreliable the functional annotation of the reads, but providing interesting results while adopted in resequencing studies [103]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=25) )

"Other important sequencing technologies are Pac Bio from Pacific Biosciences and Ion Torrent from Life Technologies [94, 96]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=25) )

"Third Generation Sequencing (TGS). Features of the emerging technology rely on the sequencing of entire single molecules of DNA, avoiding the biases introduced by the step of DNA fragmentation, PCR-based construction of cluster and phased sequencing. Moreover, TGS exploit the high catalytic rates and processivity of the DNA" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=25) )

"polymerase, enabling the sequencing of longer reads in a shorter time, when compared with the "previous" SGS technology." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

"TGS technologies are grouped into three main categories depending on the principle exploited to inspect the DNA molecule [104, 105]. Single Molecule Real Time sequencing (SMRT, Pacific Bioscience) is the first TGS technology; it is based on the observance of a DNA polymerase while it synthesizes a ssDNA molecule, using labelled nucleotides [106]. Nanopore sequencing technology inspects the DNA molecule sequence by measuring the translocation of nucleotides across a nanopores-containing membrane. Nucleotides are cleaved by the ssDNA molecule and driven through the nanopore via differential salt concentration [107]. Third category of sequencing technologies relies on the direct observation of the DNA throughout the use of advanced atomic microscopy techniques that enable the visual observation and chemical detection of the DNA molecule building blocks, simultaneously [108]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

"Metatranscriptomics is an emerging discipline based on the NGS platforms (mainly Roche and Illumina), aimed to investigate the gene expression of the whole microbial community [93]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

"information about all the potential functions a given microbial community is capable of," ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

"Indeed, metagenomics provides information about all the potential functions a given microbial community is capable of, but does not provide indication on the effective gene expression (or gene dynamics) of the microbial community under determined circumstances [94, 109]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

<i>So it's hypothetical ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26)</i> )

"indication on the effective gene expression (or gene dynamics) of the microbial community under determined circumstances" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

"metatranscriptomics enable a comprehensive annotation of the structure of the transcribed genes, highlighting their startand stop-site, and assessing the potential splicing variants (or protein isoforms) per each of the expressed gene [110]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

<i>Great for eukaryotes ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26)</i> )

"Although the sequencing technology is the same adopted in metagenomics, key differences are represented by the selective removal of the interfering nucleic acids (e.g. DNA, t-RNA, rRNA) and m-RNA to cDNA transcription prior to library production and its subsequent sequencing [111]. A comprehensive review on the RNA-seq technology is provided in [112] and [113]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=26) )

"Although of enormous importance, metatranscriptomics lack to consider the cellular organization and regulation levels governing the process of mRNA translation and protein maturation (e.g. controlled proteolysis, protein turnover, post-translational modification etc.), which greatly impact the final expression of the genome itself." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"On this basis, one could argue that metaproteomics and metabolomics (addressing the small molecules/metabolites) provide the most realistic picture of the key effectors that directly mediate the metabolic functions operated by the organisms at the specific moment of sampling [115]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"Early proteomics investigations were performed through 2DE, often complemented by spot detection and MS characterization [114]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"An improvement of this technology is represented by 2D-DIGE which employs diverse dyes in a single electrophoresis run, overcoming the issue of the scarce reproducibility among multiple 2DE runs, and enabling a direct comparison of several samples in a high-fidelity manner [114]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"Although these techniques are successful for microbial isolates, interest in the investigation of more complex samples, such as the microbiota, leads to the advent of high-performance MS platforms" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"MS is currently the dominant approach for almost all proteomics investigations. It fulfils the technological requirements for a proteomic investigation that include high throughput processing, sensitive analyte detection, wide dynamic range, mass accuracy, ability to characterise and resolve peptide sequences." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"Improvements of the actual MS technologies enable proteomics measurements that provide a wide array of information such as differential proteins expression (e.g. overtime or following experimental treatments), investigation of sub-proteomes (i.e. protein profile of the subcellular structures), post-translational modification (PTM) pattern, protein-protein interactions, and absolute protein quantitation [115]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"MS-based proteomics protocols are mainly distinct into two major branches: top-downand bottom-up-proteomics. The first is a straightforward strategy, where the complex proteins mixture is separated via liquid chromatography (LC) prior the mass spectrometry (MS) or tandem mass spectrometry (MS/MS) measurements. This protocol is particularly useful when interested in the protein identity but also in its whole molecular form and the presence of isoforms. However, analysis might be severely complicated by the range of possible protein modification (e.g. truncation, SNPs, PTM) affecting the protein mass as predicted from the genome sequence. In addition, the protocol still suffers from drawbacks related to separation and detection of large proteins (&gt;50 kDa). These limitations, coupled with the intrinsic challenge of the metaproteomes (e.g. increased sample complexity, species variability and protein homology), preclude the adoption of the top-down proteomics for the investigation of complex samples such as the microbiota [115, 116]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=27) )

"Bottom-up (or shotgun) proteomics includes additional processing steps that enable far deeper proteomics measurements. Here, proteins are primarily digested in an enzyme-specific manner (e.g. trypsin) to peptides." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Peptides are subsequently separated via LC prior to their MS/MS measurements." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Fragmentation spectrum resulting from the first MS measurement (MS1) indicate the parent ion mass spectrum of the peptides directly eluting from the LC system; whereas the tandem mass spectrum (MS2) provide insights on the fragmentation pattern and sequence information of the previous parent ions. Altogether, mass spectra are acquired and serve as a bar-code that univocally identify a given peptide [117]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Key point in the experimental proteomics protocols is represented by the efficient protein separation prior to their measurements, either via top-down or bottom-up proteomics" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"In this context, gelbased and gel-free proteomics approaches can be distinguished, depending on the use of an intermediate, gel-mediated, pre-fractionation step of the complex proteins mixture." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Gel-based proteomics is also known as the classical proteomics workflow, expecting a preseparation of the protein mixture through SDS-PAGE or 2DE." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Nowadays, a variety of offline LC systems, simplifying the complex mixtures upfront the LC-MS/MS instrument, are also employed. These perform the protein separation according to one or multiple dimensions (e.g. isoelectric point, charge, hydrophobicity, size) and are usually compatible with the advanced online separation technologies such as the ultra-high pressure liquid chromatography (UHPLC), providing a better separation of the complex proteins mixture, thus enabling improved protein detection rate [115, 122, 123]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Metabolomics provides a snapshot of the metabolites suite (metabolome) produced by a given cell, or consortia, at the moment of sampling, enabling their link back to the ongoing physiological processes [124]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"In the context of metabolomics, it is defined "metabolite" any molecule smaller than 1 kDa [125]; this includes intermediates and end-products of metabolic processes, hormones, and other signaling molecules." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"Similarly to transcriptome and proteome, metabolome is very dynamic; therefore, its investigation under diverse circumstances provide precious" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=28) )

"information on the causality of a given phenotype or, inversely, the systemic effects of specific substances (e.g. drugs) and other experimental treatments on the targeted cell lines or even the microbiota [126]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"Main steps in metabolome investigation include the pre-separation of the complex metabolites mixture, following the cell lysis and purification of the extracted metabolites. Metabolites separation is generally accomplished through HPLC, enabling for a wide range of metabolites to be analyzed at a high sensitivity." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"Gold standard for the metabolites detection is represented by MS, where, similarly to proteomics, information on the metabolite IDs and subsequent quantitation is computed back on the basis of the mass spectral fingerprint and the fragmentation pattern. Although HPLC-LC-MS is the most used approach metabolomics, nuclear magnetic resonance (NMR) spectroscopy is also widely used in analysis that specifically targets a reduced number of metabolites [124]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

<i>It would be good to review material from class ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29)</i> )

"Metatranscriptomics and metaproteomics are strongly dependent on the quality and availability of the (meta-) genomes, used as a blueprint for the mRNA and protein identification, respectively." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"Although the huge advances in the sequencing technologies, it is still difficult to obtain complete strain-specific genomes and, in the context of the microbiota evaluation, only partial population genomes are generally available." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"Commonly used metagenomics tools are MEGAN (MEtaGenome ANalyzer), IMG/M (Integrated Microbial Genomes/Metagenomes), and MG-RAST (MetaGenome-Rapid Annotation using Subsystem Technology)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"These, although based on different algorithms and a slightly different statistics, enable the taxonomic and functional annotation of the metagenomic sequences, and allow the comparative evaluation of multiple datasets" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"A typical LC-MS/MS experiments generate hundreds of thousands of fragmentation spectra. Here, the MS1 provide information on the mass-to-charge (m/z) of the intact peptide along with its amino-acidic composition whereas the MS2, providing information of the fragmentation pattern of each parent ion, determine the correct amino-acidic order." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"The peptide sequences, in" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=29) )

"turn, are assigned to specific protein IDs through a protein database-dependent search, where the measured peptides are compared with the peptides profile deriving from the in silico digestion of user-defined protein database, designated as the reference for the investigated organism or consortia." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"Clearly, this enormous amount of data cannot be manually elaborated and a wide variety of bioinformatics software and tools is nowadays available to perform the computational operations, required to translate the myriad of spectra into a meaningful output that is both concise and informative [115]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"To this purpose, commonly used search engines are OMSSA (Open Mass Spectrometry Search Algorithm), X!Tandem, MASCOT, Andromeda and SEQUEST." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"Major tasks of these tools include quality filtering of the raw MS spectra, peptidespectrum matching and scoring, protein database searching, data mining, and graphical representation of the obtained results [129]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"Similarly to metagenomics tools, proteomics/metaproteomics-committed tools are based on diverse algorithms and statistics, resulting in varying performance that, for each tool, depends on several factors such as sample complexity, separation efficiency and quality of the reference database [127]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"Three main strategies can be employed to obtain a suitable reference database." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"The first option is to use the metagenome sequence of the exact same sample or arising from a comparable community/habitat. This is considered the best option since has been associated with an improved identification rate" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"A second choice is to create an artificial, in silico metagenome (pseudometagenome) that includes genome sequences of microbial species supposed to be present in the microbial community of interest. This strategy has been demonstrated to be effective [130], but a priori knowledge of the taxonomic complexity is required." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"The third option expects a survey search before the "official" main search. The survey is conducted against a publicly available database (e.g. NCBInr) and the list of identified proteins is exported either unfiltered or weakly filtered, in order to obtain a comprehensive database to use in the later main search. This approach enables the use of small-sized database (leading to a higher identification rate) and avoids the bias introduced with the manual selection of the microorganisms. Further taxonomic levels may also be implemented from publicly available repositories [127]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"On the attempt to overcome the total dependency of metaproteomics from the metagenomics sequence availability, bioinformatics tools are currently being developed to perform a new bioinformatics approach termed "de novo sequencing "of the uncharacterized proteins. The rationale behind this promising approach expect the de novo calculation of the peptide 19" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=30) )

"Chapter 1 sequences on the basis of the parental ion masses and their fragmentation pattern provided respectively from the MS1 and MS2 of the tandem mass spectrum. This will provide the peptide sequence of peptides that are not included in the current metagenomes as well as enable insights on the PTM and protein polymorphisms. Although of great interest, the accuracy and speed of this groundbreaking approach still need substantial improvements to facilitate its widespread implementation in the "routinely" practice [115, 131]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

<i>This is a wild idea ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31)</i> )

"Detailed metaproteomic characterization of a given microbial community requires a quantitative assessment of the identified protein repertoire. The methods applied for protein quantification are distinguished as label-basedor label-free approach depending on the need to label the target proteins to gain quantitative information." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Because of the independence of any cultivation step and the ability to estimate protein amount without any labelling procedure, the label-free quantification is the most suitable approach for metaproteomic studies [127, 132, 133]. However, this requires high reproducibility LC-MS/MS data." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Indeed, quantification is based on either spectral counting (SC) or signal intensity (SI). Quantification through SC relies on the assumption that the number of fragment spectra mirrors the abundance of the peptide the spectra belong to. Normalized Spectra Abundance Factor (NSAF) is considered an improved version of the SC, since it estimates the protein abundance considering the effect of the protein size, thus avoiding the overestimation of larger proteins. Main software adopted for SC and NSAF quantification are Scaffold, Proteo IQ, APEX, CENSUS etc; a comprehensive review of these tools is provided in [127, 132, 133]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"The SI-based quantification relies on the peptide ion abundance at specific retention times to estimate protein abundance. Although it is performed by a variety of software (e.g. MaxQuant, Elucidator, Open MS, SIEVE), the label-free quantification provided by MaxQuant is currently the most used, enabling an accurate detection and comparison of changes in the abundance of both highand lowabundant proteins [127]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Key points of the MaxQuant software reside in the integrated algorithms performing the process of "ID transfer" and "peak normalization" leading to improved peptide ID and accurate relative protein quantification, respectively [134]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Another important task of the bioinformatics in the field of metaproteomics concern the annotation of identified proteins into predicted functional classes and clustering of the metaproteome dataset into functional groups." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Data repositories can be distinguished on the basis of the annotation type. UniProt KB, for instance, is further divided into Swiss-Prot and TrEMBL. The first comprises the only entries whose annotation has been reviewed and manually annotated; whereas, TrEMBL entries are not reviewed and "only" automatically annotated." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Other data repositories commonly used are Kyoto Encyclopedia of Genes and" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=31) )

"Genomes (KEGG), Gene Ontology (GO), BioCarta" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=32) )

"Moreover, KEGG integrates all these information and organize them as biochemical pathways that fall into seven main groups, namely metabolism, genetic information processing, environmental information processing, cellular process, organismal system, human diseases and drug development." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=32) )

"Diverse other software and web-based application are nowadays available for this purpose, such as Web MGA, Cytoscape, IPath, DAVID and others (as reviewed in [127]) that retrieve and integrate functional annotation from diverse sequence repositories to provide a more comprehensive functional annotation of the queried metaproteomics dataset." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=32) )

"The microbial community harboring the GIT of chickens has been investigated mainly in relation to its spatial microbial diversity [56, 135, 136]. A variety of studies is also focused on the investigation of the avian gut microbiota following dietary supplementation of antimicrobials [47, 137], probiotics [50, 138, 139], and high-energy feed formulations [140]. Only a handful of studies investigated the effect of Ca, P and MP supplementation on the chicken gut microbiota composition [141, 142]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=32) )

"In the first study, diets varying in the amount of supplemented CaP and/or MP were fed to growing broilers, aimed to investigate whether the differences in the diet composition are also reflected in the gut-associated bacterial community, both in terms of composition and activity of the microbial community. A metaproteomic approach was used to assess the dietary-induced changes in the composition of the bacterial fraction of the chickens crop and ceca" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=33) )

"Depicted bacterial community was compared with the total bacterial community designated, for the same sections and dietary treatments, via 16S rRNA gene sequencing. Functional features of the microbial community of chickens crop and ceca identified on protein level, at diverse dietary treatments, were assigned through the functional categorization of the identified protein profiles into bioinformatic data repositories." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=33) )

"Videnska P, Faldynova M, Juricova H, Babak V, Sisak F, Havlickova H, Rychlik I: Chicken faecal microbiota and disturbances induced by single or repeated therapy with tetracycline and streptomycin. BMC Vet Res 2013, 9(1):30." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=38) )

"Danzeisen JL, Kim HB, Isaacson RE, Tu ZJ, Johnson TJ: Modulations of the Chicken Cecal Microbiome and Metagenome in Response to Anticoccidial and Growth Promoter Treatment. PloS one 2011, 6(11):e27949." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=38) )

"Polansky O, Sekelova Z, Faldynova M, Sebkova A, Sisak F, Rychlik I: Important Metabolic Pathways and Biological Processes Expressed by Chicken Cecal Microbiota. Appl Environ Microbiol 2015, 82(5):1569-1576." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=38) )

"Zoetendal EG, Rajilic-Stojanovic M, de Vos WM: High-throughput diversity and functionality analysis of the gastrointestinal tract microbiota. Gut 2008, 57(11):1605-1615." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=42) )

"Compared to the structural information obtained by metagenomic analyses, metaproteomic approaches provide, in addition, functional data about the investigated microbiota." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=49) )

"In general, integration of the main Omics-technologies (genomics, transcriptomics, proteomics and metabolomics) in live science promises highly detailed information about the specific research object and helps to understand molecular changes in response to internal and external environmental factors." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=49) )

"The microbiotas of livestock like chicken, pig and ruminants are becoming a focus of interest for veterinaries, animal nutritionists and microbiologists. While pig is more often used as an animal model for human-related studies, the rumen microbiota harbors a diversity of enzymes converting complex carbohydrates into monomers which bears high potential for biotechnological applications." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=49) )

"Despite cultivation being a sound technique to characterize the physiological properties of microorganisms [1], there are severe drawbacks in using this as a tool for characterizing bacterial communities. Typically, the culture media do not resemble in situ conditions and in some cases the carbon richness is higher than the substrates found in situ, allowing the growth of only a small fraction of the community while suppressing other members [2]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=50) )

"The inconsistency between in situ and cultivable diversity has resulted in the widespread use of culture-independent molecular approaches [3, 4]. Microbial community profiling methods (16S ribosomal RNA gene based approaches) have become important tools to characterize microbial communities and the interactions between the microorganisms present in the GIT." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=50) )

"Overall, the analysis of the microbiota is important to improve animal nutrition strategies and animal health. This knowledge can be used to modulate the microbiota to reduce antibiotic treatments and, in the case of ruminants, to inhibit the formation of emission gases" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=50) )

"Fig. 1. Workflow of possible methods to study the structure and function of the microbiota in farm animals." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=51) )

"Both systems have unique features, such as short paired-end reads (max. 2 × 300 bp) with Illumina vs. long read length (600-800 bp) with 454. The latter one is more feasible in terms of shotgun sequencing studies (see below) [5, 6], while Illumina provides barcoding strategies and bigger data sets that are more favorable to analyze hundreds of samples in targeted sequencing projects [6]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=51) )

"Two other techniques, that were not frequently applied for metagenomic studies of animal microbiota, are the Ion Torrent (Life Technologies) and the PacBio (Pacific biosciences)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=51) )

"The gene of choice to analyze the phylogenetic composition of a microbial community is the 16S rRNA gene" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"There are a couple of techniques in molecular ecology, such as fingerprinting methods, microarrays and fluorescence in situ hybridization which use the 16S rRNA gene as a target molecule." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"Nowadays the total diversity of a microbiological sample is analyzed preferably by pyrosequencing of the 16S rRNA gene, obtained by amplification of extracted DNA." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"Subsequent to pyrosequencing, quality filtering and denoising processes have to be applied. The reads should be checked for chimeras and clustered to operational taxonomic units (OTU) in order to assign the respective taxonomies to the sequences. There is a diverse range of bioinformatic tools available in free software platforms such as Mothur, QIIME, RDP pipeline, LIBSHUFF, UniFrac and MEGAN that support data analysis and convert data to formats that can later be used in statistical packages like R, Metastats or Primer-E. A detailed overview of the methods can be found in several review papers [5, 8]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"overview of the methods can be found in several review papers [5, 8" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"These pipelines should be used with special care as it is not only important to make sense of all the raw data, but also to ensure that the final picture is a direct reflection of the original raw data collected and thus of the original community structure of the sample. The output data reveal ecological indices, relative abundance values of the identified taxa and enable a pre-selection for a targeted quantitative PCR (real-time PCR) approaches if necessary." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"In addition to the phylogenetic structure of the community, the analysis of encoded and expressed metabolic pathways is the second objective. Metagenomic or metatranscriptomic data are obtained by shotgun pyrosequencing of the total DNA and cDNA, respectively." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"Reads have to be quality filtered, assembled to contigs, binned and assigned to taxonomies and possible gene functions. As the assembly requires sequence reads with appropriate length, so far 454 pyrosequencing was the method of choice as it produces reads up to 800 bp (see above). Due to the progress in data generation and bioinformatic processing Illumina pyrosequencing is recently used as well. Several tools are available for the annotation of open reading frames on the contigs, MG-RAST [9], MEGAN5 [10], IMG/M [11], Metarep [12] and MicroScope [13]. CAMERA portal [14] was shut down in July 2014. These tools can also be used for metabolic pathway reconstruction. This is usually done based on the KEGG database [15] or the subsystem classification of SEED [16]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=52) )

"In addition to metatranscriptomic studies, the community activity can be assessed based on expressed proteins and formed metabolites." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"Metaproteomic studies investigate the protein inventory of a specific sample at a certain point of time [17]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"The first key step is to find an optimized sample preparation protocol to avoid co-extraction of eukaryotic proteins and to get a purified protein sample." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"The following workflow depends on the available technical equipment [18]. In a gel-based approach, proteins are separated and proteolytically digested into peptides followed by a one-dimensional liquid chromatography directly coupled to the mass spectrometric analysis (LC-MS/MS)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"In a gel-free approach, peptides are prepared by insolution digestion directly in the protein mix. Peptides are separated by two-dimensional LC and measured by MS/MS analysis." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"The protein identification is the second big challenge as it is highly depending on the available sequence database which can either be used from public resources or sample-specific sequences. An overview of available bioinformatic tools and workflows are given in [18, 19]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"An overview of available bioinformatic tools and workflows are given in [18, 19" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"The coverage of metaproteomic studies of complex microbial samples, such as feces or rumen contents, is still low. Since there is a high species diversity and cell density in these types of samples, only abundant proteins are identified while rare species, that may have important metabolic functions, are missed. Targeted proteomic approaches, like selective reaction monitoring (SRM) can be used to specifically detect and quantify proteins of interest [20]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"NMRand MS-based methods are available and their application to detect defined groups of metabolites is reviewed by Xie et al. [21]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"The chicken intestinal environment comprises a vast and diverse assemblage of microorganisms living not as single species populations, but rather in complex communities comprising multiple species that include animal and human pathogens. Intricate networks of interactions between the microorganisms and their environment shape the respective communities and are important for animal welfare and food safety reasons. The chicken GIT consists of more than 900 species of bacteria. This diverse microbiota helps not only the breakdown and digestion of food but also plays an important role concerning the growth and health of the host [22]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=53) )

"In the past, the chicken GIT microbial community was studied by culture-based methods. These studies discovered that 10-60% cecal bacteria can be cultured [23, 24] and about 45% could be assigned to the genus level [22]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"The profiles of the different gut sections are nowadays studied using cultivation-independent methods like clone libraries [25-27], denaturing gradient gel electrophoresis (DGGE) [28], temperature gradient electrophoresis (TGGE) [27], terminal restriction fragment length polymorphism (T-RFLP) [4, 29-31], quantitative PCR (qPCR) [32], microarrays [33], next-generation sequencing [34-39] and metaproteomics [40]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"Regarding the research in microbial ecology of the chicken GIT, several studies focused on the influence of diet [25, 31, 32, 41], antimicrobial feed additives [29, 34] host genotype [38, 42], gender [38], spatial microbial diversity [25, 28, 31, 43], age [28], development and temporal microbial variations [22, 26, 28]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"Sklan et al. showed that the different sections of the chicken GIT are highly inter-connected [44]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"Sklan et al. showed that the different sections of the chicken GIT are highly inter-connected [44]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"However, because of the high diversity within each section, it has been suggested to analyze them as separate ecosystems [28]. It was demonstrated that the microbial communities colonizing the GIT of chicken benefit the host [29, 31, 36]. Nevertheless, two recent studies revealed that this colonization can also harm the host [35, 45]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"This is a moment of great importance regarding the establishment of the microbial communities." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"Although the colonization of the chickens by maternally derived bacteria is low, some studies postulated that the microbial community structure of the small intestine settles within two weeks. Older studies showed that cecal bacteria need longer time to develop [40, 46]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"The gut is colonized by commensal, transient and pathogenic microorganisms. Commensal microorganisms are beneficial to the host as they provide amino acids, short-chain fatty acids and vitamins [40]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"The chicken gut is divided in three upper segments: crop, proventriculus and gizzard. The crop is a food storage muscular pouch related to the breakdown of starch and the fermentation of lactate. Digestion starts in the proventriculus while the gizzard grinds food. Because of its lower pH and fermentation activity, the gizzard functions as microbial barrier. Similar microbial communities were found in the crop and gizzard. Lactobacilli, facultative and microaerophilic bacteria are the most dominant bacteria present in this two segments. Other abundant species belonged to Clostridiaceae, Enterococcus and in the case of the crop also Bifidobacterium and" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=54) )

"Enterobacteriaceae (Fig. 2) [25, 46, 47]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"The small intestine is relatively long and has a constant diameter. It consists of three parts: the duodenum, jejunum and ileum where the nutrient absorption and food digestion occurs" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"Due to the low pH, pancreatic and bile secretions, the bacterial density in the duodenum is comparably low" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"Besides Lactobacillus as the main colonizer of the jejunum (reaching coverage of up to 99%), Streptococcus was identified as well." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"Amit-Romach et al. has shown that the relative proportion of Lactobacillus spp. in duodenum and jejunum increases within age [48]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"The chicken's ileum harbors Lactobacillus in higher abundance (&gt; 68%) and in lower abundances Streptococcus, Enterobacteriaceae and Clostridiaceae [28, 43]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"Lu et al. demonstrated that during all different stages of microbial community development in the ileum Lactobacilli were dominant [26]. This gut section is also known to be colonized by novel butyrate producing bacteria that may play an important role regarding the availability of nutrients, absorption rate and chicken performance [47]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=55) )

"Chickens have two caeca which are important for recycling urea, the absorption of water, and digestion of cellulose, starch and polysaccharides. These two fermentation chambers have the highest bacterial density and are colonized by obligate anaerobes like Clostridium, Bacteroidetes, and Bifidobacterium (Fig.2) [42]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=56) )

"Recently, 16S rDNA amplicon pyrosequencing studies estimated a bacterial population of about 700 species [39]. This wealth of microorganisms makes the caeca an important study site and a reservoir rich in unknown and uncultured microorganisms and pathogens [30, 39, 46, 47]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=56) )

"Qu et al. proved that mobile DNA elements are the cause of functional microbiome evolution and that horizontal gene transfers and the metavirulomes of cecal microbiomes were related to the host environment [49]. A metagenomic analysis of the chicken caecum using the Illumina MiSeq 2000 system revealed a relatively high proportion of sequences encoding glycosyl hydrolases that were identified by sequence comparison with carbohydrate active enzymes (CAZY) database (Fig. 3) [39]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=56) )

"More than 200 genes of non-starch polysaccharide degrading enzymes were identified indicating a great potential for xylane degradation compared to a lower cellulolytic potential in the caeca." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=56) )

"Fig. 3. Abundance of glycoside hydrolase (GH)-families in metagenomes of bovine rumen and chicken caecum. The percentage of each GH-group relative to the total number of GH-families identified in each metagenomic dataset is shown grouped according to major activity [90]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=57) )

"Chicken feces samples are colonized by Lactobacillus, Clostridium, Faecalibacterium, Ruminococcus, Bacillus, Eubacterium, and Fusobacterium ( Fig. 2)" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=58) )

"Here the microbiota is not stable and it has been proposed that these fluctuations are related to the emptying of the previous gut sections [25]. A recent study in meta-analysis of the avian gut microbiota showed that genes related to cytokine receptors and cell adhesion grouping into "signaling molecules and interaction" were less present in fecal samples indicating a lower potential of host/bacteria interactions [50]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=58) )

"The only metaproteomic study using a chicken fecal sample identified about 3487 proteins in total [40]. Bacterial proteins mainly belonged to Lactobacillus and Clostridium. Gene ontology analyses showed that the majority encodes for stress-related proteins like chaperons and proteases as well as enzymes involved in glycolysis [40]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=58) )

"The inclusion of penicillin in diets increases the body weight of chickens and also the Firmicutes to Bacteroidetes ratio in caeca. These effects might be caused by a reduction of the weight of the small intestine and the thickness of the gut wall, increasing the absorption of nutrients" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=58) )

"The addition of the antibiotics tetracycline and streptomycin also induces a rapid shift in microbial community, increasing the prevalence of Lactobacillales and Enterobacteriales in fecal samples." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=58) )

"For the first time, the present study analysed dietary effects on the protein inventory of the microbiome in crop and ceca of broilers." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=77) )

"We performed quantitative label-free metaproteomics by using 1- D-gel electrophoresis coupled with LC-MS/MS to identify the structural and functional changes triggered by diets supplied with varying amount of mineral phosphorous (P) and microbial phytase (MP)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=77) )

"Within the ceca section, proteins of Bacteroidaceae were more abundant in the P-supplied diets, whereas proteins of Eubacteriaceae decreased with the P-addition." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=77) )

"Classification of the identified proteins indicated a thriving microbial community in the case of P and MP supplementation, and stressed microbial community when no P and MP were supplied. Data are available via ProteomeXchange with identifier PXD003805." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=77) )

"The chicken gastrointestinal tract (GIT) microbiome comprises more than 900 species of microorganisms [1]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"Microbial composition changes longitudinally and radially along the gastrointestinal tract, since each intestinal section has its own characteristic "microenvironment" suitable for a better colonization by specific microbial species [2, 3]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"To date, several studies investigated the chicken microbiota based on cultivation and 16S rRNA gene analysis (for reviews see [2, 4-6])." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"for reviews see [2, 4-6]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"In a recent study, Sergeant et al. [9] analyzed the metagenome of the cecal content from a single bird in order to assess the bacterial phylogenetic distribution and its potential activity. Another recent study investigated the microbiome of two pooled fecal samples based on 16S rRNA gene sequencing and metaproteomics in order to understand the metabolic processes in the gut of a healthy chicken [11]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"Polansky et al. investigated the cecal microbiome of chickens at different ages, including 1-week-old chickens after inoculation with cecal extracts from hens of different ages, on the attempt to explain colonization patterns and predict the most promising probiotic genera for cecal colonization of newly hatched chickens [12]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"Despite the extensive and heterogeneous ensemble of published studies, in-depth investigations about the possible functional changes of the microbiome of the chicken gut challenged with different dietary treatments are missing." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"Phosphorous (P) is an essential macro element involved in a multitude of physiological processes such as bone development, growth and productivity of livestock." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"In plant seeds, organic phosphorous is mainly present as phytate, the salt of myo-inositol 1,2,3,4,5,6-hexakis (dihydrogen phosphate) (phytic acid; InsP6). InsP6 is only partially digestible for many monogastric animals such as chicken [13, 14]. Here, P bioavailability can be improved by phytases, phosphatases responsible for phytic acid dephosphorylation and release of P, available for the animal absorption in the intestine." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"Since endogenous phytases in chicken are less efficient and their activity depends on dietary phosphorous and cholecalciferol [15, 16], standard chicken diet formulation requires supplementation of mineral P and microbial phytase (MP) in order to reduce the problem of low endogenous phytase activity and ensure adequate P" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=78) )

"provision to the animal [17]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

"Additional phytase dosage also helps to reduce the need for supplementation of mineral P in the feed formulation." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

"This leads to a consequent reduction of P excretion which has a great significance for the ecological problem of water eutrophication and saving of P, a limited resource of global importance." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

"To the best of our knowledge, no studies are published so far investigating the functional changes of the bacterial community inhabiting different GIT sections and correlating it to the different effects of broiler diets either supplemented with mineral P and/or MP." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

"Witzig and colleagues (2015) characterized changes in bacterial phylogenetic compositions along the GIT of broilers fed with different mineral P and MP supplemented diets by T-RFLP and 16S rRNA gene sequencing [20]. They showed an effect of mineral P and MP towards the abundance of certain Lactobacillus spp. especially in the crop and a possible influence of MP in the cecal community." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

<i>Find a paper to use a as base and add to it ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79)</i> )

"Here, we used this feeding experiment to analyze proteins extracted from crop and cecal bacterial communities to obtain indications for possible functional changes in the microbiome of these gut sections. A label-free quantitative (LFQ) metaproteomic approach was used for the assessment of the protein phylogenetic composition and abundance. Discussion on the detected functional pathways and the change of abundance of certain proteins in the diverse dietary treatments is given." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

"Samples were obtained from an animal experiment described in detail by Zeller et al. [14]. A schematic overview of the experimental workflow is provided in Figure A in S2 file. A total of 1,140 unsexed broiler hatchlings (Ross 308) were obtained from a local hatchery (Brüterei Süd GmbH and Company KG, Regenstauf, Germany). One hundred and eighty birds were housed in 18 pens (10 animals each) to investigate tibia mineralization, details on [14]. The remaining 960 animals were housed in 48 pens (20 animals each). On day 15 of age, broiler chickens were" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=79) )

"assigned to six experimental treatments (8 pens/treatment) until day 25 of age" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"From six pens per treatment, four animal each, were chosen for DNA-based investigation of the chicken GIT microbiota as detailed in [20]. Out of these, 4 animals each from two pens per treatment (48 animals in total) were chosen for the metaproteomic investigations." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"Remaining animals were used for investigation on inositol phosphate degradation [14]. The contents of crop as well as of cecal sections were separately collected and homogenized on a pen basis, yielding two pooled cecal and crop samples, respectively, per dietary treatment, for a total of twenty-four samples (Figure A in S2 file)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"All animals were fed a commercial broiler starter diet until day 14 of age. At day 15 the experimental diets were fed, all consisting of a basal diet [21] containing adequate levels of all nutrients, according to the recommendations of the German Society for Nutritional Physiology (GfE), with the exception of mineral P and calcium" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"Three diets contained P exclusively deriving from plant sources (BD-), whereas in another three diets additional P was supplied as mono-calcium phosphate (BD+). Diets of BD - and BD+ groups were further supplemented with 0, 500 and 12500 U/kg respectively of an Escherichia coli microbial phytase product (Quantum Blue, EC 3.1.3.26, supplied by AB Vista, Marlborough, UK), allowing a further distinction between MP0 and MP+ (MP500 and MP12500) diets." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"Twenty four pooled samples were kept on dry ice during their transport to the laboratory and stored at -80°C until their analysis. After thawing at 4°C, bacterial cells were separated by using a previously described method [22] with modifications. Briefly, aliquots of 0.5 g of pooled samples were resuspended by vortexing in 15 mL washing buffer (50 mM Na2HPO4, 0.1% Tween 80, [pH 8.0]). Samples were then incubated for 10 min in a sonication bath (amplitude 50%, 0.5 cycle), shaken for 20 min in a reciprocal shaker at 100 oscillations/min and centrifuged at low speed (200 x g, for 15 min, 4°C). Supernatant containing the bacterial cells was collected in a 50 mL tube and the remaining pellet was subjected to further 4 rounds of the whole protocol, for a total of five rounds." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"Bacterial cells in the pooled supernatant were pelleted by centrifugation at 15,000 x g for 30 min at 4°C and subjected to protein extraction." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=80) )

"3.3.4 Protein extraction, quantitation, digestion" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=81) )

"MS/MS instrument was set to positive ion mode. Full scan was acquired in the mass range from m/z 300 to 1650 in the Orbitrap mass analyzer at a resolution of 120,000 followed by HCD fragmentation of the 12 most intense precursor ions. High resolution MS/MS spectra were acquired with a resolution of 30,000. The target values were 3*106 charges for the MS scans and 1*105 charges for the MS/MS scans with a maximum fill time of 25 ms and 45 ms," ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=81) )

"respectively. Fragmented masses were excluded for 30 s after MS/MS. Spectra de-noising was performed prior to peptide identification by considering the only top 12 peaks in a window of 100 Da width." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=82) )

"To reduce the false discovery rate of peptide identification and for a better evaluation of protein abundance a two-step approach for bioinformatics data analysis was chosen. Briefly, proteins identified in the first database-dependent search are used to build a second sample-specific database, used for the second database-dependent search. The smaller size of the latter database enabled for a high coverage of the metaproteome as well as a reduced number of false positive inference [25, 26]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=82) )

"Acquired raw data were at first processed using Thermo Proteome Discoverer software (v.1.4.1.14), Mascot (v. 2.4) and searched against NCBI-nr bacteria and chicken databases (release July 12th, 2014) in order to export a protein fasta database." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=82) )

"Methionine oxidation was set as variable modification and carbamidomethylation of cysteine as fixed modification. Default settings of the software were kept, these include protein grouping with peptide confidence set on "high" and delta Cn of 0.1" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=82) )

"In the second process, exported protein fasta file from the first search was subsequently used as in-house database for the peaks alignment and mass re-calibration in the first step of the MaxQuant search. MaxQuant software (v.1.5.1.2) set on LFQ modality was used for peptide identification and protein IDs inference. In the second step of MaxQuant analysis, raw data were independently searched against UniProtKB databases (release October 2014) bacteria (UniProt ID 2, 18976242 entries) and chicken (UniProt ID 9031, 82439 entries)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=82) )

"The mass spectrometry proteomics data are publicly available in the ProteomeXchange Consortium via the PRIDE [27] partner repository with the dataset identifier PXD003805. Further insights on the identified peptides and their implication for protein IDs inference, are provided in the S2 and S3 Tables." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=82) )

"Phylogenetic information were inferred on basis of the protein description outputted from the MaxQuant searches. These, in turn, are gathered from the protein annotation of the chosen database (i.e. UniProt KB)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"LFQ abundances from MaxQuantś output results were subjected to statistical analysis by using Primer6 v.6 statistical software (PRIMER-E, Plymouth, UK). Principal coordinate analysis (PCoA) was calculated on the basis of the Bray-Curtis similarity matrix [28]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"Statistical difference between diet treatments was calculated by performing analysis of variance with permutations (PERMANOVA" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"Statistically different treatments were then subjected to SIMPER analysis in order to isolate proteins responsible for dissimilarity between pairs of groups with a cut-off threshold of 99.99% [29]" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"Selected proteins were functionally classified into COG and KEGG categories by using WebMGA on-line tool [30] with an e-value cutoff of 10-3 and exclusively considering the best hit." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"Cladograms visualizing the dietary effects on the structure of the chickenś crop and cecal microbiome were drawn using the Galaxy on-line tool. It implements the computation of the Linear Discriminant Analysis [21] between the technical triplicates of each experimental diet group (n = 2). Here, the Kruskal-Wallis test is performed to check whether differences between the experimental diets are statistically significant (p &lt;0.05" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"Despite using a protocol to analyze microbial proteins, it is a common phenomenon that eukaryotic proteins are always co-extracted and processed during the metaproteomics workflow [33-35]. In this study, a total of 248 and 405 host proteins were identified in the crop and cecal section, respectively, which were investigated in order to highlight the possible presence of specific transporters or any other activity related to the adaptation to the changing dietary conditions." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"Functional classification of the identified proteins was performed by categorization of their abundance intensity into COG classes and KEGG pathways" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=83) )

"No statistical differences in chicken protein abundances were observed between the diets in both GIT sections. The reason may be found in the relatively low protein numbers, which were obtained by the sample preparation protocol, which favored prokaryotic cells [22]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"The used sample preparation protocols yielded a total of 381 and 1,719 bacterial proteins for crop and ceca sections respectively, with 3.1% of the total proteins shared among both sections." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"The relatively low identification rate in the crop section is probably explained by the low bacterial abundance typical of this section and the high amount of feed residues, which were co-extracted [37]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"Details on the number of identified proteins and peptides in each sample, as well as a general overview of the abundance intensities of proteins in both GIT sections along with their grouping into KEGG biochemical pathways, are summarized in Figure B in S2 file and S5 Table." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"In the crop section, 38.9% of the total variation was observed in the PCoA1, where metaproteomes of the samples without mineral P supplementation clustered together and drift apart from the samples supplemented with mineral P (p = 0.043, Fig 1A)" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"PCoA analysis of the cecal samples showed a clustering of the microbial proteins from birds fed with mineral Psupplied diets and a separation from treatments without mineral P supplementation, which was not significant (p &gt; 0.05, Fig 1B)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"A very clear distinction was shown between the MP-containing diets that clustered apart from the MPlacking diets (p = 0.008)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=84) )

"The PCoA analysis of cecal samples (Fig 1B) also showed that the pairs of biological duplicates were close together, with an average similarity of 72.2% (ranging from 69.0% to 84.5%), meaning that observed differences represent a "true" biological difference induced by our experimental treatments." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=86) )

"The phylogenetic composition of the bacterial proteins extracted from crop and ceca was determined based on the abundance values of the proteins belonging to each bacterial family" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=86) )

"The abundance of each bacterial family in respect to the experimental diets is shown for the crop (Fig 2A) and cecal (Fig 2B) samples" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=86) )

"Only bacterial families showing a discriminative effect (p &lt; 0.05) with respect to the diets were considered and ranked according to their contribution in the different experimental treatments." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=86) )

"In addition, a phylogenetic assessment was done considering only families expressing at least one phylogenetic marker protein (i.e. highly conserved proteins employed as marker for phylogenetic analysis) [38-41] (S6 Table)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=86) )

"In general, phylogenetic distribution of the crop metaproteomes showed a reduced bacterial diversity and a high inter-individual diversity among all dietary treatments causing the unpaired scattering of the biological duplicates in the PCoA analysis (Fig 1A)" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"In accordance with other studies on the chickenś GIT microbiota composition (for reviews see [2, 4, 6]), proteins belonging to Lactobacillaceae were the most abundant regardless of the diets (Figure C in S2 file)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"The number of proteins belonging to Veillonellaceae increased on average in BD+ diets (36%) when compared with the BDdiets (23%; Figure C panel A in S2 file, Fig 2A)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"Veillonellaceae has been often associated to fiber digestion and short chain fatty acid production [43, 44], therefore its increase in P-supplied diets suggests a potential beneficial effect for the animal growth and performance [45]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

<i>Implied improvement ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88)</i> )

"With the exception of Veillonellaceae and a few other families commonly found in the chickenś upper GIT (Propionibacteriaceae, Erysipelotrichaceae, Eubacteriaceae, Clostridiaceae) [46], other minor bacterial families identified in this study such as Nocardiaceae, Gordoniaceae, Bradyrhizobiaceae, Rhizobiaceae, Moraxellaceae, Desulfovibrionaceae and Pseudomonadaceae are more likely to be found in environmental samples [47, 48]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"However, all these families were also found in the gut microbiome of humans or other animals [46, 49-51], therefore their presence in the crop microbiome may be either attributed to an intake from the environment (e.g. with the feed) or such bacterial families can be considered as common members of the cropś microbial fraction." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"The DNA-based phylogenetic analysis of the same crop samples analyzed by Witzig et al. resembled also the predominance of Lactobacillaceae and a decrease of them concomitant to MP supplementation [20]. Other minor bacterial families were either not detected in the DNAbased study or in the present one." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"Phylogenetic distribution of the cecal microbial community (Fig 2B) showed a higher phylogenetic diversity of the identified proteins in all dietary treatments and a change in the composition depending on the diets. Proteins belonging to Bacteroidaceae showed an average abundance of 14.9% in mineral P-supplied diets regardless of the presence of MP compared to 8.4% abundance in the samples from the diets without mineral P supplementation. Conversely, proteins belonging to Eubacteriaceae were more abundant in all diets without mineral P supplementation (4.6%) than with mineral P supplementation (3.9%). Proteins belonging to Ruminococcaceae were more abundant in the diets with increasing amounts of MP (25.4% in the MP+ diets vs. 11.5% in the MP0 diets) whereas proteins of Lactobacillaceae showed a" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=88) )

"contrary abundance (5.2% in MP+ vs. 27.8% in MP0 diets; Fig 2B, Figure C panel B in S2 file). The increased abundance of Bacteroidaceae in the BD+ diets as well as the decrease of Lactobacillaceae due to the MP supplementation is in line with the results of the DNA-based analysis performed by Witzig et al. [20]." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=89) )

"However, no increased abundance of Bacteroidaceae due to the dietary MP addition was observed and no Erysipelotrichaceae members were found in the present metaproteomic investigation." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=89) )

"Nevertheless, a higher bacterial diversity in the microbiome of crop and ceca sections was highlighted with the present approach. The discrepancy in the results from the same samples is imputable to the different methods. The amplification steps of T-RFLP and amplicon pyrosequencing analyses, including a possible primer bias and the presumed overestimation of taxa with a higher number of 16S rRNA genes, are probably the reason of the reduced heterogeneity found in the microbiota composition" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=89) )

"On the other hand, this step allows a higher sensitivity to target minor bacterial families which are missing in the metaproteomic approach due to the lack of genomic sequences." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=89) )

"Similarly, inconsistencies between DNAand protein-based microbiota assessment was described by Tang et al. [11]. They also showed that the correlation between the potential and active bacterial community is not always possible since the species identification in proteomics was different to that of 16S rRNA gene sequencing." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=89) )

"PERMANOVA analysis of the total datasets of both sections was done to check for significant differences between diets. Crop samples showed that the only pair of treatments MP0 - MP12500 across P factor were statistically significant (p = 0.048)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=89) )

"The entries of the statistically different treatments were subjected to SIMPER analysis, using a strict cut-off threshold (99.99%), to identify single proteins that caused the dissimilarity between treatment groups. These proteins were functionally classified by grouping them into COG categories (Fig 3)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=90) )

"In MP12500 metaproteome, lipid metabolism (I) was uniquely identified and three categories were more abundant in comparison with MP0 such as translation, ribosomal structure and biogenesis (J), carbohydrate transport and metabolism (G) and amino acid transport and metabolism (E) (Figure E in S2 file)." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=90) )

"Protein data of the cecal samples showed that proteins belonging to translation, ribosomal structure and biogenesis (J) were more abundant (23%) in the BDsamples than in the BD+ ones (14%) (Fig 3). In contrast, proteins of the carbohydrate transport and metabolism (G) group were more abundant, increasing from 16% in the BDto 24% in the BD+ diets" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=90) )

"Based on these results, our initial hypotheses that the diets will affect the functional profile of the bacterial communities was accepted. Specifically, mineral P-available diets (BD+; MP+) stimulate a "productive bacterial community" where bacterial resources are focused on complex anabolic functions; while the microbial community present at low mineral P diets (BD- ; MP0) is concerned in the stress response mechanisms, suggesting that the mineral P limitation affecting the host health status is also reflected as stress factor for the gut microbiota. This idea seems to be in accordance with a study of Tang et al., which correlated the expression of proteins involved in metabolic processes of carbohydrate, alcohol and proteins to a thriving microbial community [11]. Protein folding has been linked to the microbial stress response to the high temperature of the chicken body. In our results, the thriving condition is maintained by the adequate nutrients supplementation, while the main stress factor is represented by the lack of P. T" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=91) )

"The metabolic potential of the chickenś fecal microbiota as assessed by Singh et al. [52] shows that the abundance of the "carbohydrate metabolism" SEED subsystem was stable between the metagenomes of low and high feed conversion rate chickens whereas SEED subsystems related to sulphur metabolism and motility/chemotaxis were statistically different." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=91) )

"The abundance intensities of the proteins, responsible for significant dissimilarities between experimental treatments in the two GIT sections, were additionally grouped according to KEGG biochemical pathways (Figure F in S2 file). In both sections, major biochemical pathways were identified in all experimental treatments, but only a low number of proteins within these pathways was shared between the different treatments, suggesting a diverse overall activity of" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=91) )

"the microbial communities depending on the fed diets" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=92) )

"Additionally, differences in the abundance level of the common proteins across the diets were observed, indicating a probable modulation of the highly conserved functions of the microbiota on the attempt to shape an adequate response to the changing environment. Specifically, the higher abundance of the KEGG Orthologous system [53] for "ribosome" (KO 03010) and "aminoacyl-t-RNA biosynthesis" (KO 00970) observed in crop MP12500 samples, as well as the higher abundance of "ABC transporters" (KO 02010) registered in crop MP0 samples, suggest an increased metabolic activity, and an overall effort of the bacterial community to maximize the P uptake respectively for the MP+ and MP0 samples (Figure F in S2 file). Similar trend was observed in the cecal samples. The high-expression of several KOs such as "glycolysis/gluconeogenesis" (KO 00010, BD+ samples) and "starch and sucrose metabolism" (KO 00500, MP+ samples) indicate a productive bacterial community and a coexistence of the bacterial members in a thriving microenvironment." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=92) )

"Several KOs were found to be overrepresented in the MP0 treatments such as 'pyrimidine metabolism' and 'arginine and proline metabolism' (KO 00330). Their abundances decreased with the addition of MP, resulting in a more favorable ratio for the MP0 treatment (Fig 4 panel B and C). Conversely, 'lysine biosynthesis' as well as 'oxidative phosphorylation' registered an overrepresentation in MP0 in the only pair MP0/MP500, while a further MP addition (MP0/MP12500) led to a slight overrepresentation in the MP12500 sector." ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=92) )

"Identified proteins were sorted into KEGG metabolic maps (S7 Table) in order to confirm the previous COG results and give further support to our hypothesis that experimental diets affect the protein expression of the microbial community" ([Link](zotero://open-pdf/library/items/G2SD5DFS?page=93))