---
title: dehauOmicsTechnologiesPoultry2022
author: Ben Lorentz
date: '2022-11-07'
slug: dehauomicstechnologiespoultry2022
categories: []
tags:
  - one tag
  - another tag
meta_img: images/image.png
description: Description for the page
---


# Omics technologies in poultry health and productivity - part 1: Current use in poultry research
#### (2022) - Tessa Dehau, Richard Ducatelle, Filip Van Immerseel, Evy Goossens
- **Link**:: https://doi.org/10.1080/03079457.2022.2086447
- **DOI**:: 10.1080/03079457.2022.2086447
- **Links**:: 
- **Tags**:: #paper
- **Cite Key**:: [@dehauOmicsTechnologiesPoultry2022] 

### Abstract
In biology, molecular terms with the suffix “-omics” refer to disciplines aiming at the collective characterization of pools of molecules derived from different layers (DNA, RNA, proteins, metabolites) of living organisms using high-throughput technologies. Such omics analyses have been widely implemented in poultry research in recent years. This first part of a bipartite review on omics technologies in poultry health and productivity examines the use of multiple omics and multi-omics techniques in poultry research. More specific present and future applications of omics technologies, not only for the identification of specific diagnostic biomarkers, but also for potential future integration in the daily monitoring of poultry production, are discussed in part 2. Approaches based on omics technologies are particularly used in poultry research in the hunt for genetic markers of economically important phenotypical traits in the host, and in the identification of key bacterial species or functions in the intestinal microbiome. Integrative multi-omics analyses, however, are still scarce. Host physiology is investigated via genomics together with transcriptomics, proteomics and metabolomics techniques, to understand more accurately complex production traits such as disease resistance and fertility. The gut microbiota, as a key player in chicken productivity and health, is also a main subject of such studies, investigating the association between its composition (16S rRNA gene sequencing) or function (metagenomics, metatranscriptomics, metaproteomics, metabolomics) and host phenotypes. Applications of these technologies in the study of other host-associated microbiota and other host characteristics are still in their infancy.

### Notes
<b>Extracted Annotations (8/12/2022, 5:08:52 PM)</b> 

"Integrative multi-omics analyses, however, are still scarce." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=2) )

"microbe- (or pathogen-)host interactions for improving knowledge on disease pathogenesis and resistance" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=2) )

"Deblais et al., 2020" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=2) )

"host genetics (reviewed by Zampiga et al., 2018)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=3) )

"intestinal microbiota in an attempt to gain insight in the role of the microbiota in gut health and performance (for review see Upadhyaya et al., 2019)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=3) )

"potential future implementation of the omics tools in the daily routine of the poultry industry." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=3) )

"find genes or genetic variants that are associated with specific phenotypes, such as disease resistance or animal performance (Tuite &amp; Gros, 2006)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=3) )

"Moreover, the microbiome is essential for many biological processes including, amongst others, nutrient acquisition, immune development and pathogen exclusion (Shang et al., 2018)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=3) )

"holo-omics (Nyholm et al., 2020)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=3) )

"Deblais et al., 2020" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=4) )

"disease resistance (Cheng et al., 2013), productivity (Zampiga et al., 2018) and biomarker discovery (de Meyer et al., 2019), and understanding of the pathogenesis and host response to infectious (Deblais et al., 2020) and non-infectious challenges (Zampiga et al., 2021)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=4) )

"For example, as protein function is mediated and altered by post-translational modifications (e.g. phosphorylation, complex formation, post-translational processing of pre-proteins), correlations between gene expression levels and protein expression levels of cellular proteomes are relatively low (Washburn et al., 2003; Olivier et al., 2019)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=4) )

"host disease resistance, and host fertility, with special emphasis on how multi-omics approaches might further contribute to these research fields." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=4) )

"Therefore, genomic information alone is not enough to unravel the biological host-pathogen interaction, and future research could benefit from the use of multiple omics." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=5) )

"Functional omics, such as transcriptomics, proteomics and metabolomics, connect genome to gene functions and are needed to identify factors responsible for the successful or unsuccessful infection by a pathogen (Deblais et al., 2020)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=5) )

"Deblais et al., 2020" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=5) )

"While many papers characterize themselves as multiomics studies, they are reporting the simultaneous use of different omics without performing a real effort to integrate the different data layers, which is what multi-omics or integrative omics should aim to do." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=5) )

"only one study has integrated transcriptomic and metabolomic data to understand how the liver responds under chronic heat stress and has highlighted multiple pathways which are affected under these conditions (Jastrebski et al., 2017)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=5) )

"Jastrebski et al., 2017" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=5) )

"The diversity of the chicken GI microbiota is largely influenced by the age of the birds, the location in the digestive tract, the environment, and the diet (Oakley et al., 2014; Shang et al., 2018; Rychlik, 2020)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"Oakley et al., 2014; Shang et al., 2018; Rychlik, 2020)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"for review see Upadhyaya et al., 2019)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"Caecal bacterial taxa such as Faecalibacterium prausnitzii, Ruminococcus genus, Lachnospiraceae family and Bacteroides genus, are associated with good bird performance or low feed conversion ratio (FCR) (Stanley et al., 2012, 2013, 2014, 2016; Singh et al., 2012; Johnson et al., 2018)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) ) ^f77bd9

"can degrade complex indigestible carbohydrates present in chicken feed, such as cellulose and hemicellulose, into end metabolites that can be utilized by the bird, thereby providing additional energy to the bird" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"short chain fatty acids (SCFA), such as butyrate, which is beneficial for gut health as it provides energy to epithelial cells, stimulates cell proliferation, reinforces the gut barrier and may inhibit certain pathogens (Guilloteau et al., 2010)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"reported for the genus Lactobacillus, even if strains belonging to this genus are commonly used as probiotics in poultry. This might suggest that different strains within this same genus have different effects on feed efficiency and performance (Stanley et al., 2016; Yan et al., 2017; Johnson et al., 2018)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"it appears that hens are efficient donors of Bacteroidetes, Actinobacteria, Selenomonadales and Faecalibacterium, but not of most important Grampositive bacteria inhabiting the gut such as Clostridiales or Lactobacilli, which appear to come mostly from the environment (Kubasova et al., 2019a)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"can efficiently colonize the caecum of newly hatched chicks after a single dose, while bacteria commonly used as probiotics such as Bacillus, Lactobacillus or Enterococcus do not (Kubasova et al., 2019b)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=6) )

"because commercially hatched chicks have no contact with the parent hens, the initial gut microbiota composition is greatly affected by the environment, and the microbiome composition of young birds is highly variable between studies (Rychlik, 2020)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"In contrast to the high variability in microbial populations, the functional pathways of the microbiome seem more stable and conserved across studies (Qi et al., 2019)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"One of thefirst examples combined both 16S rRNA gene and metagenomic sequencing to assess the effect of anticoccidial and antibiotic treatments on the caecal microbiome (Danzeisen et al., 2011)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"Pathways for non-starch polysaccharides (NSPs) utilization, SCFA production and hydrogen consumption were identified (Sergeant et al., 2014" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"Thefirst gene catalogue of the chicken gut microbiome was created in 2018, covering all intestinal compartments, and generated from metagenomic data (Huang et al., 2018)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"acquired antimicrobial resistance genes in poultry farms (Luiken et al., 2020; Feng et al., 2021)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"As the same genera are present in the avian gut and human gut, the major metabolic characteristics of those bacterial species will be comparable, and overall functional predictions of the poultry microbiome are believed to be reliable (Rychlik, 2020)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"Indeed, multiple studies have observed large discrepancies between the active microbiome derived from metaproteomics, and the total microbiome obtained through 16S rRNA gene sequencing (Tang et al., 2014; Borda-Molina et al., 2016, 2018)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) )

"In one example, metabolomics was employed to understand the caecum microbiome-host interaction during Salmonella infection in young layers (Mon et al., 2020)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7) ) ^c90692

<i>This might be good to evaluate or ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=7)</i> )

"arginine and proline metabolism were enriched after Salmonella Enteritidis infection, which was consistent with host caecal tonsil RNA-seq data showing the upregulation of arginine-associated pathways at 3 days post-infection." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=8) )

"Another integrative study combined metagenomics and metatranscriptomics to reveal the abundance, diversity, and" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=8) )

"expression of antibiotic-resistance genes (ARGs) in chickens and other species (Wang et al., 2020)" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=8) )

"Wang et al., 2020" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=8) )

"Some studies have combined 16S rRNA gene sequencing, performed on gut content, with transcriptomics or proteomics on host gut tissue, to investigate associations between microbiota and the intestinal immune function (Oakley &amp; Kogut, 2016; Willson et al., 2018; Rodrigues et al., 2020)." ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=8) )

"Oakley &amp; Kogut, 2016; Willson et al., 2018; Rodrigues et al., 2020" ([Link](zotero://open-pdf/library/items/REIYLVKJ?page=8))